## Introduction {.page_break_before}

On January 21, 2020, the World Health Organization (WHO) released its first report concerning what is now known as the Coronavirus Disease 2019 (COVID-19) [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf].
This infectious disease came to international attention on December 31, 2019 following an announcement by national officials in China describing 44 cases of a respiratory infection of unknown cause.
The first known cases were located in Wuhan City within the Hubei province of China, but the disease spread rapidly throughout China and subsequently around the world.
At the time of the WHO's first situation report [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf], 282 confirmed cases had been identified.
Most of these cases were in China, but 1-2 exported cases had also been identified in each of several neighboring countries (Thailand, Japan, and the Republic of Korea).
One week later, 4,593 confirmed cases had been identified, spanning not only Asia, but also Australia, North America, and Europe [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200128-sitrep-8-ncov-cleared.pdf].
On March 11, 2020, WHO formally classified the situation as a pandemic [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf].
On April 4, 2020, the WHO reported that the global number of confirmed cases had surpassed one million [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200404-sitrep-75-covid-19.pdf].
{{csse_deaths}} COVID-19 deaths had been reported worldwide as of {{csse_date_pretty}} (Figure @fig:csse-deaths).

![
**Cumulative global COVID-19 deaths since January 22, 2020.**
Data are from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University [@https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series].
]({{csse_deaths_figure}} "Global COVID-19 deaths"){#fig:csse-deaths secno=1}

Several review articles on aspects of COVID-19 have already been published.
These have included reviews on the disease epidemiology [@doi:10.1016/j.molmed.2020.02.008], immunological response [@doi:10.1016/j.immuni.2020.05.002], diagnostics [@doi:10.1126/scitranslmed.abc1931], and pharmacological treatments [@doi:10.1016/j.immuni.2020.05.002; @doi:10.1001/jama.2020.6019].
Others [@doi:10.1038/d41591-020-00026-w] and [@doi:10.1001/jama.2020.12839] provide narrative reviews of progress on some important ongoing COVID-19 research questions.
However, research on these topics is proceeding so quickly that any static review is likely to quickly become dated.
Our goal was to consolidate information about the virus in the context of related viruses and to synthesize rapidly emerging literature centered on the diagnosis and treatment of COVID-19.
We developed infrastructure centered on the Manubot publishing system [@doi:10.1371/journal.pcbi.1007128] to manage hundreds of contributions from the community to this living, scholarly document.
We designed software to develop living figures like Figure @fig:csse-deaths, which automatically update using external data sources.
Our primary goal is to sort and distill informative content out of the overwhelming flood of information [@doi:10.1038/s42254-020-0175-7] and help the broader scientific community become more conversant this critical subject.
Thus, our approach has been to develop a real-time, collaborative effort that welcomes submissions from scientists worldwide into this ongoing effort.
This document represents the first snapshot, which aims to reflect the state of the field as of August, 2020.
We plan to refine and expand this document until technologies to mitigate the pandemic are widely available.

### Interdisciplinary Context

Collaboration across several broad areas of research is critical, as different areas provide different information and context necessary to understanding the virus and disease.
This review provides a biological perspective on the SARS-CoV-2 virus and efforts to identify biomedical developments targeted at COVID-19.
We provide only brief summaries of two other important perspectives on this pandemic: epidemiology and public health.
Epidemiological analysis has investigated properties related to infection and transmission such as when infected individuals are contagious, how the virus is transmitted between individuals, the range of symptoms associated with infection and/or contagiousness in different individuals, and how rapidly the virus propagates between individuals, etc.
Other areas of public health that address resource availability, inequity, human behavior, and other components that influence people's exposure to pathogens and ability to manage illness will also be critical to mounting a global response to the pandemic.
Public health strategies to manage epidemics include anticipation and early detection of emerging diseases (aided by rapid development of diagnostics), containment (using strategies to test, trace and isolate such as widespread testing, contact tracing, quarantining and isolation), control and mitigation of spread (including social distancing), and elimination and eradication.
An effective public health response requires response coordination, disease surveillance and intervention monitoring, risk communication and health education (including education about personal protective equipment and hand hygiene and containment of “infodemics” of false information), non-pharmaceutical measures to prevent and reduce transmission, and health interventions such as vaccines and other pharmaceuticals that can reduce transmission, curb morbidity and mortality, and mitigate the effects on health systems and other sectors of society [@isbn:978-92-4-156553-0].
<!--I think non-pharmaceutical interventions are already in categories above (tracing, social distancing, lockdowns etc)-->
These fields intersect with the topics addressed in this manuscript because they both inform and benefit from relevant biotechnological developments (e.g., accurate and fast diagnostic testing).
However, the present analysis focuses less on human and social factors and more on the basic biology of infection, diagnosis, and recovery. 

### Rapid Characterization of SARS-CoV-2 

Because the SARS-CoV-2 virus was not identified until COVID-19 was already a major health concern, scientists and physicians have used information about related viruses and diseases to guide both investigations of the biology of the virus and disease and the development of diagnostic, prophylactic, and therapeutic approaches.
<!-- To Do: When/how did they figure out it was a coronavirus?-->
The first genome sequence of the SARS-CoV-2 virus was produced in January 2020 [cite].
Genomic analyses have made it possible to place the virus phylogenetically relative to coronaviruses infecting humans and other animals and to study diversity within the viral species.
This rapid contextualization of the virus has made it possible to develop hypotheses rapidly about how the virus might infect cells, transmit between people, and respond to existing therapeutics.
<!--To Do for HMR: Talk about Oxford vaccine group and virology attempts to have an open-ended response ready?--> 
Understanding coronaviruses as a taxon and how humans respond to viral threats is foundational to understanding the pathogenesis of SARS-CoV-2 and the COVID-19 pandemic. 

#### Origin and Evolution

The hypothesis best supported by the collected genomic data is that SARS-CoV-2 has a zoonotic origin.
Though the intermediate host serving as the source for the zoonotic introduction of SARS-CoV-2 to human populations has not yet been identified, the SARS-CoV-2 virus has been placed within the coronavirus phylogeny through comparative genomic analyses.
Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
Human SARS-CoV-2 is genetically closer to bat RATG13 than to any other known virus, including human SARS-CoV.
The similarity between SARS-CoV-2 and RATG13 is as high as 96.2%, while the similarity between SARS-CoV-2 and SARS-CoV is only 79% [@doi:10.1038/s41586-020-2012-7; @doi:10.1093/nsr/nwaa036].
Nevertheless, some fragments between SARS-CoV-2 and RATG13 have a difference of up to 17%, suggesting a complex natural selection process during zoonotic transfer.
While the S fragment is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253], there is no consensus about the origin of S in SARS-CoV-2.
It could potentially be the result either of recombination or coevolution [@doi:10.1093/nsr/nwaa036; @doi:10.1126/sciadv.abb9153].
Regardless, while there is substantial support for the zoonotic origin of SARS-CoV-2, a lot remains unknown, causing issues regarding the SARS-CoV-2 evolutionary pattern [@doi:10.1038/s41591-020-0820-9, @doi:10.1038/s41586-020-2012-7]. <!-- TODO: I don't understand what this sentence is conveying -->

After zoonotic transfer, SARS-CoV-2 continued evolving in the human population [@doi:10.1186/s12967-020-02344-6].
The SARS-CoV-2 mutation rate is moderate compared to other RNA viruses [@doi:10.1016/j.meegid.2020.104351], and that restricts SARS-CoV-2 evolution pace.
Nevertheless, genomic data show some significant statistical evidence of ongoing evolution.
There are two known variants of the spike protein that differ by a single amino acid at position 614 (G614 and D614), and there is evidence that G614 has already become dominant over D614 by June 2020 [@doi:10.1016/j.cell.2020.06.043].
While there is a hypothesis that this genomic change increased the SARS-CoV-2 infectivity and virulence, there is no data yet to support or discard this hypothesis reliably [@doi:10.1016/j.cell.2020.06.040].
One study [@doi:10.1016/j.meegid.2020.104351] has reported more than 198 recurrent mutations using a dataset of 7666 curated sequences showing the dynamic of SARS-CoV-2 evolution.
Much uncertainty remains about SARS-CoV-2's evolutionary trends though it is evident that SARS-CoV-2 exhibits moderate potential for further evolution.

The numerous mutations [@doi:10.1186/s12967-020-02344-6; @doi:10.1093/nsr/nwaa036; @doi:10.1016/j.meegid.2020.104351] observed in individual SARS-CoV-2 genome sequences signal transmission patterns and can be used during outbreak investigations.
Similar mutations observed in several patients may indicate that the patients belong to the same transmission group.
The tracking of SARS-CoV-2 mutations is recognized as an essential tool for controlling outbreaks that may provide valuable insights into the paths of SARS-CoV-2's spread [@doi:10/d5mg].
Several studies used phylogenetic analysis to determine the source of local COVID-19 outbreaks in Connecticut (USA), [@doi:10.1016/j.cell.2020.04.021], the New York City area (USA) [@doi:10.1126/science.abc1917], and Iceland [@doi:10.1056/NEJMoa2006100].
There is an ongoing effort to collect SARS-CoV-2 genomes throughout the COVID-19 outbreak, and as of August 7, 2020, more than 78,000 genome sequences have been collected from patients. <!-- TODO: consider auto-updating this -->
The sequencing data can be found at GISAID [@https://www.gisaid.org/], NCBI [@https://www.ncbi.nlm.nih.gov/sars-cov-2/], and COVID-19 data portal [@https://www.covid19dataportal.org/].

#### Human Immune Reponse to Viral Threats

Understanding the fundamental organization of the human immune response to viral threats is critial to understanding the varied response to SARS-CoV-2.
The human immune system utilizes a variety of innate and adaptive responses to protect against the pathogens it encounters.
The innate immune system consists of barriers, such as the skin, mucous secretions, neutrophils, macrophages, and dendritic cells.
It also includes cell-surface receptors that can recognize the molecular patterns of pathogens.
The adaptive immune system utilizes antigen-specific receptors that are expressed on B and T lymphocytes.
These components of the immune system typically act together; the innate response acts first, and the adaptive response begins to act several days after initial infection following the clonal expansion of T and B cells [@doi:10.1016/j.jaci.2005.09.034].

After a virus enters into a host cell, its antigen is presented by major histocompatibility complex 1 (MHC 1) molecules and is then recognized by cytotoxic T lymphocytes.
One of the main immune responses that contribute to the onset of Acute Respiratory Distress Syndrome (ARDS) in COVID-19 patients is the cytokine storm, which causes an extreme inflammatory response due to a release of pro-inflammatory cytokines and chemokines by immune effector cells.
In addition to respiratory distress, this mechanism can lead to organ failure and death in severe COVID-19 cases [@doi:10.1016/j.jpha.2020.03.001].
Research in response to prior outbreaks of hCoV-borne infections, such as SARS and MERS, have provided a strong foundation for hypotheses about the pathogeneis of SARS-CoV-2 as well as potential diagnostic and therapeutic approaches. 

### Role of the COVID-19 Review
<!-- Rough draft, but basically: distill signal from noise to help accelerate the discovery process-->

With the worldwide scientific community uniting during 2020 to investigate SARS-CoV-2 and COVID-19 from a wide range of perspectives, findings from many disciplines are relevant on a rapid timescale to a broad scientific audience.
Additionally, many findings are published as preprints, which are available prior to going through the peer review process.
As a result, centralizing, summarizing, and critiquing new literature broadly relevant to COVID-19 can help to expedite the interdisciplinary scientific process that is currently happening at an advanced pace. 
We are particularly interested in providing background to the development of diagnostic, prophylactic, and therapeutic approaches to COVID-19.
Two major concerns within diagnosis include the detection of current infections in individuals with and without symptoms, and the detection of past exposure without an active infection.
In the latter category, identifying whether individuals can develop or have developed sustained immunity is also a major consideration.
The development of high-throughput, affordable methods for detecting active infections and sustained immunity will be critical to understanding and controlling the disease.
The identification of interventions that can mitigate the effect of the virus on exposed and infected individuals is a significant research priority.
Some possible approaches include the identification of existing pharmaceuticals that reduce the severity of infection, either by reducing the virus' virulence (e.g., antivirals) or managing the most severe symptoms of infection.
Due to the long timeline for the development of novel pharmaceuticals, in most cases, research surrounding possible pharmaceutical interventions focuses on the identification and investigation of existing compounds whose mechanisms may be relevant to COVID-19.
Other foci of current research include the identification of antibodies produced by survivors of COVID-19 and the development of vaccines.
Understanding the mechanisms describing host-virus interactions between humans and SARS-CoV-2 is thus critical to identifying candidate therapeutics.

Thus, in this review, we seek to consolidate information about efforts to develop strategies for diagnosis and therapeutics as new information is released by the scientific community.
We include information from both traditional peer-reviewed scientific literature and from preprints, which typically have not undergone peer review but have been critically evaluated by the scientists involved in this effort.
The goal of this manuscript is to present preliminary findings within the broader context of COVID-19 research and to identify the broad interpretations of new research, as well as limitations to interpretability.
