## Introduction {.page_break_before}

### General Background

On January 21, 2020, the World Health Organization (WHO) released its first report concerning what is now known as the Coronavirus Disease 2019 (COVID-19) [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf].
This infectious disease came to international attention on December 31, 2019 following an announcement by national officials in China describing 44 cases of a respiratory infection of unknown cause.
The first known cases were located in Wuhan City within the Hubei province of China, but the disease spread rapidly beyond Wuhan within China and subsequently around the world.
At the time of the WHO's first situation report [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200121-sitrep-1-2019-ncov.pdf], 282 confirmed cases had been identified, primarily in China, but also 1-2 exported cases had been identified in several neighboring countries (Thailand, Japan, and the Republic of Korea).
One week later, 4593 confirmed cases had been identified, spanning not only Asia, but also Australia, North America, and Europe [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200128-sitrep-8-ncov-cleared.pdf].
On March 11, 2020, WHO formally classified the situation as a pandemic [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200311-sitrep-51-covid-19.pdf].
On April 4, 2020, the WHO reported that the global number of confirmed cases had surpassed one million [@url:https://www.who.int/docs/default-source/coronaviruse/situation-reports/20200404-sitrep-75-covid-19.pdf?sfvrsn=99251b2b_4].

**Global COVID-19 deaths since January 22, 2020.**
Data from the COVID-19 Data Repository by the Center for Systems Science and Engineering at Johns Hopkins University [@https://github.com/CSSEGISandData/COVID-19/tree/master/csse_covid_19_data/csse_covid_19_time_series].
]({{csse_deaths_figure}} "Global COVID-19 deaths"){#fig:csse-deaths}

Some review articles on aspects of COVID-19 have already been published and may be of great usefulness.
These have included reviews on the disease epidemiology [@doi:10.1016/j.molmed.2020.02.008], immunological response [@doi:10.1016/j.immuni.2020.05.002], and pharmacological treatments [@doi:10.1016/j.immuni.2020.05.002; @doi:10.1001/jama.2020.6019].
However, research on these topics is proceeding so quickly that any static review is likely to quickly become dated.

In this review, we seek to consolidate information about the virus in the context of related viruses and to synthesize what is known about the diagnosis and treatment of COVID-19 and related diseases.
Further, we aim to amplify the true signal out of the vast noise produced by thousands of publications on the topic [@doi:10.1038/s42254-020-0175-7].
We will critique, sort and distill informative contents out of the overwhelming flood of information and help the larger scientific community to be better educated on this critical subject affecting all of us.
This is a real-time, collaborative effort that welcomes submissions from scientists worldwide.

### About Coronaviruses

#### Coronavirus Genome and Structure

Coronaviruses (CoVs; order *Nidovirales*, family *Coronaviridae*, subfamily *Orthocoronavirinae*) are enveloped viruses with some of the largest viral RNA genomes.
Coronavirus genomes range from 27 to 32 kilobases (Kb) in length.
The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp [@doi:10/ggjr43].
There are several fundamental genomic characteristics shared by all viruses in the *Nidovirales* order, including enveloped, non-segmented, and positive-sense (ssRNA+) genomes and a large number of non-structural genes originating through ribosomal frameshifting [@doi:10.1007/978-1-4939-2438-7_1].
Genome organization is highly conserved within the order [@doi:10.1007%2F978-1-4939-2438-7_1].
Coronavirus genomes are comprised of a replicase gene that is two-thirds of the genome.
This polypeptide is translated into 16 non-structural proteins (nsp1-16, except in Gammacoronaviruses, where nsp1 is absent), which form replication machinery used to synthesize viral RNA [@doi:10.1002/jmv.25681].
The remaining third of the genome encodes structural proteins, including the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins.
Additional accessory genes are sometimes present between these two regions, depending on the species or strain.
<!-- To do: Add a figure showing the structure of the genome.-->
<!-- Also, everything focuses on S! Should we also talk about others?-->

Coronaviruses are classified into four genera: alpha, beta, delta and gamma coronaviruses.
Among them, alpha and beta coronaviruses infect mammalian species, gamma coronaviruses infect avian species and delta coronaviruses infect both mammalian and avian species [@doi:10.1146/annurev-virology-110615-042301].
The viruses were initially subdivided into these genera based on antigenic relationships of the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins but are now divided by phylogenetic clustering.
[this section may be better as a figure panel + legend]
Phylogenetic analysis of a PCR amplicon fragment from five patients along with the total virus genome of 29.8 kilobases indicates that the virus is a novel betacoronavirus belonging to the B lineage, also known as sarbecovirus.
The sarbecovirus lineage also includes the human SARS coronavirus [@doi:10/ggjs7j].

Most coronaviruses are considered zoonotic viruses with little to no transmission observed in humans.
A major group of coronaviruses include human coronaviruses (HCoVs) strains associated with multiple respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals [@doi:10.1086/377612].
Approximately one-third of common cold infections in humans is attributable to four out of six previously known human coronaviruses (HCoV-229E, HCoV-NL63, HCoV-OC43 and HCoV-HKU1) that are globally circulating in the human population [@doi:10.1038/s41579-018-0118-9;@doi:10.3390/diseases4030026].
In the past two decades, however, highly pathogenic human coronaviruses have been identified, including the severe acute respiratory syndrome coronavirus 1 (SARS-CoV-1) and the Middle East respiratory syndrome coronavirus (MERS-CoV) although both infections were confined to specific geographic regions [@doi:10.1038/nrmicro.2016.81;@doi:10.1038/s41579-018-0118-9].
The current pandemic of COVID-19, caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), represents an acute and rapidly spreading global health crisis with symptoms for reported cases ranging from mild to severe or fatal [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html] and including outcomes such as acute respiratory distress, acute lung injury and other pulmonary complications.
The possibility of asymptomatic cases is also being investigated.
The transmission and mortality rate estimations of COVID-19 remain to be determined.

#### Transmission and Evolution

SARS-CoV-2 is a novel human coronavirus, following SARS-CoV and MERS-CoV to become the third coronavirus causing severe pneumonia in humans to emerge in the 21st century [@doi:10.1016/bs.aivir.2018.01.001]. 
The evolutionary origins of the SARS-CoV-2 virus are not yet fully understood.
Genomic analyses and comparisons to other known coronaviruses suggest that SARS-CoV-2 is unlikely to have originated in a laboratory -- either purposely engineered and released, or escaped -- and instead evolved naturally in an animal host [@doi:10.1038/s41591-020-0820-9].
Among known coronaviruses, SARS-CoV-2 has the closest overall sequence similarity to RaTG13 (~96%) found in a *Rhinolophus affinis* bat [@doi:10.1038/s41586-020-2012-7], while the receptor binding domain (RBD) is highly similar to that of viruses found in pangolins [@doi:10.1101/2020.02.19.950253].
This suggests that SARS-CoV-2 may have originated in viral reservoirs of similar hosts; however, current evidence cannot discriminate an origin of the virus before or after zoonotic transfer to humans [@doi:10.1038/s41591-020-0820-9].

Most coronaviruses are considered zoonotic viruses with little to no transmission observed in humans.
A major group of coronaviruses include human coronaviruses (HCoVs) strains associated with multiple respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals [@doi:10.1086/377612].
Approximately one-third of common cold infections in humans is attributable to four out of six previously known human coronaviruses (HCoV-229E, HCoV-NL63, HCoV-OC43 and HCoV-HKU1) that are globally circulating in the human population [@doi:10.1038/s41579-018-0118-9;@doi:10.3390/diseases4030026].
In the past two decades, however, highly pathogenic human coronaviruses have been identified, including the severe acute respiratory syndrome coronavirus 1 (SARS-CoV-1) and the Middle East respiratory syndrome coronavirus (MERS-CoV) although both infections were confined to specific geographic regions [@doi:10.1038/nrmicro.2016.81;@doi:10.1038/s41579-018-0118-9].

The current pandemic of COVID-19, caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), represents an acute and rapidly spreading global health crisis with symptoms for reported cases ranging from mild to severe or fatal [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html] and including outcomes such as acute respiratory distress, acute lung injury and other pulmonary complications.
The possibility of asymptomatic cases is also being investigated.
The transmission and mortality rate estimations of COVID-19 remain to be determined.
In general, respiratory viruses like coronavirus can have multiple routes of person-to-person transmission including droplet transmission (i.e. inhalation for cough, sneeze), aerosol transmission (i.e. virus suspended in air), and contact transmission (i.e. contact with oral, nasal, and eye mucous membranes).
Other modes of transmission, such as through touching surfaces or objects and then touching mucous membranes, can also be investigated.
While droplet-based and contact were initially considered to be the primary modes of SARS-CoV-2 transmission [@doi:10.1038/s41368-020-0075-9], as additional information has emerged, aerosol transmission has become a growing concern [@10.1126/science.abc6197].
Other aspects of transmission to investigate are the relationship between infectiousness and virus shedding with disease period or symptoms and also the proportions of cases that are attributable to various types of transmission events, such as transmission between relatives, nosocomial transmissions, and other possible types of interactions.
Some information about these characteristics of transmission are available for the highly pathogenic coronaviruses SARS-CoV and MERS-CoV [@doi:10.1016/B978-0-12-385885-6.00009-2; @doi:10.1038/nrmicro.2016.81].
For SARS-CoV-2 it is still being investigated whether, in addition to being spread by people who show symptoms,  the virus can be transmitted by people who do not show symptoms [@url:https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html], such as during the pre-symptomatic stage of infection or in people with asymptomatic infections (a presentation which sometimes occurs in other infectious diseases but is still being investigated for SARS-CoV-2).

#### Cell Entry and Replication

Coronavirus virions are spherical with diameters ranging from 100 to 160 nanometers (nm).
The virion is made up of a lipid envelope in which peplomers of 2-3 spike (S) glycoproteins are anchored, creating a distinctive "crown" shape for which the family of viruses was named [@isbn:0781760607; @doi:10.1128/JVI.80.3.1302–1310.2006].
The replication process is initiated by the viral spike protein: first, the virus binds to a host cell, and then the viral membrane fuses with the endosomal membrane to release the viral genome into the host cytoplasm.
The spike of SARS-CoV is structured in three segments: the ectodomain, a transmembrane anchor, and an intracellular tail [@doi:10.1128/JVI.02615-14].
The ectodomain contain two subdomains known as the S1 and S2 subunits, with the S1 (N-terminal) subunit guiding the binding of the virus to a host cell receptor and the S2 (C-terminal) subunit guiding the fusion process [@doi:10.1128/JVI.02615-14].
<!-- Is it worth going into more detail about siganl peptides and secretory pathways? Not sure whether this is relevant information.-->
The ectodomain forms the crown-like structures on the viral membrane, with the S1 domain forming the head of the crown and containing the Receptor Binding Motif (RBM) that covalently bonds with a host receptor, while the S2 domain forms the stalk that supports the head [@doi:10.1038/nrmicro2090].
Among betacoronaviruses, some are known to bind to CEACAM1, Neu 5,9 Ac2, and ACE2 [@doi:10.3390/v4061011].

After the virus binds to a host cell receptor, the viral and plasma membranes must then fuse.
Many coronaviruses are cleaved at the S1-S2 boundary and remain covalently bound until fusion to the host cell is achieved, with the S1 subunit stabilizing the S2 subunit during the membrane fusion process [@doi:10.1016/j.cell.2020.02.058].
This priming of the S protein is induced by the binding of cellular proteases [@doi:10/cf8chz; @doi:10.1016/j.antiviral.2020.104792].
Angiotensin-Converting Enzyme 2 (ACE2) and Transmembrane Serine Protease 2 (TMPRSS2) have been identified as a prime receptor and a critical protease, respectively, facilitating SARS-CoV/CoV-2 entry into a target cell [@doi:10.1038/nature02145; @doi:10.1126/science.abb2507; @doi:10.1128/JVI.01542-10; @doi:10.1016/j.cell.2020.02.052; @doi:10.1128/JVI.02232-10].
This finding has led to a hypothesized role for ACE2 and TMPRSS2 expression in determining which cells, tissues and organs are most likely to be infected by SARS-CoV-2.
A second cleavage site within S2 is thought to activate _S_ for fusion by inducing conformational change [@doi:10.1016/j.cell.2020.02.058].
Electron microscopy suggests that in some coronaviruses, including SARS-CoV and MERS-CoV, a six-helix bundle separates the two subunits in the postfusion conformation, and the unusual length of this bundle facilitates membrane fusion through the release of additional energy [@doi:10.1146/annurev-virology-110].
However, research in SARS-CoV suggests that, in addition to fusion with the plasma membrane, viruses may also be taken up by cells through endocytosis [@doi:10.1038/cr.2008.15].
<!-- This is very interesting and seems like it could be relevant to virulence (?), but I don't know enough cell biology to expand on it. Another reference that seems useful is doi.org/10.1146/annurev-biochem-060208-104626-->
Once the virus enters a host cell, the replicase gene is translated and assembled into the viral replicase complex, which can synthesize dsRNA genome from the genomic ssRNA(+).
Finally, the dsRNA genome is transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes [@doi:10.1007/978-1-4939-2438-7_1; @url:https://viralzone.expasy.org/30?outline=all_by_species].

Thus, most of the viral entry process is determined by the viral genome, but the initial step, when the virus binds to a host receptor, also depends on the virus being able to recognize the host receptor.
Coronaviruses such as SARS-CoV, SARS-CoV-2, and MERS-CoV may therefore be engaged in an evolutionary arms race with the human immune system [@doi:110.1101/2020.05.13.093658; @doi:10.1101/2020.05.14.09613; @doi:10.1186/1743-422X-10-304].
Coronaviruses bind to a range of host receptors [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009], with binding conserved only at the genus level [@doi:10.1146/annurev-virology-110615-042301].
The S1 domain, which contains the RBM, evolves more rapidly than the S2 domain [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009].
However, even within the S1 domain there is variability in conservation.
The receptors in the N-terminal domain (NTD) evolve more rapidly than those in the C-terminal domain (CTD) [@doi:10.1016/j.jmb.2020.04.009]. 
Both the NTD and CTD are involved in receptor binding and can function as receptor binding domains (RBDs) to bind proteins and sugars [@doi:10.3390/v4061011], but RBDs in the S1-NTD typically bind to sugars, while those in the S1-CTD recognize protein receptors [@doi:10.1146/annurev-virology-110615-042301].
Viral receptors show higher affinity with protein receptors than sugar receptors [@doi:10.1146/annurev-virology-110615-042301], which suggests that positive selection on or relaxed conservation of the S1-NTD preferentially might reduce the risk of a deleterious mutation that would prevent binding.
<!--also positive selection on MHC/HLA? Seems worth mentioning somewhere, possibly here?...-->
Cell entry by the virus is a critical component to pathogenesis and therefore an important process to understand when examining possible therapeutics.
Pathogenesis is made up of three major components: entry, replication, and spread [@isbn:0-9631172-1-1 {chap. 45}]

### COVID-19: Infection and Presentation

<!--Fill in from other PR when merging! -->

### Approaches to Understanding COVID-19

Scientific characterization of the SARS-CoV-2 virus and of the COVID-19 disease it causes is critical to controlling the current pandemic.
Several broad areas of research interact with each other, offering different pieces of information critical to understanding the virus and disease.
A comprehensive understanding of the epidemic must unify basic scientific and medical research with public health and biotechnology.

#### Science & Medicine

Understanding how the virus functions and interacts with the host is foundational to understanding pathogenesis and disease progression and to identifying available and novel approaches to treatment.
Therefore, the fields of virology, immunology, and molecular biology are fundamental to characterizing SARS-CoV-2 and COVID-19.
These topics can be approached using a range of techniques, including characterization of the host response from the cellular to systems level.
Contextualizing SARS-CoV-2 in relation to other viruses that infect humans and other animals can further serve to elucidate the reaction of the human host to viral exposure.
This information, when combined with an understanding of the biology of pharmaceutical and medical interventions, can guide new approaches to treatment.

#### Public Health

One necessary component of determining how to manage the outbreak is to understand epidemiological factors related to the transmission of the SARS-CoV-2 virus.
These can include characteristics such as when infected individuals are contagious, how the virus is transmitted between individuals, the range of symptoms associated with infection and/or contagiousness in different individuals, and how rapidly the virus propagates between individuals, etc.
The development of diagnostic tools is critical to this goal.
Accurate diagnoses on a large scale is necessary to collect the data needed to develop epidemiological models.
Other areas of public health that address resource availability, inequity, human behavior, and other components that influence people's exposure to pathogens and ability to manage illness will also be critical to mounting a global response to the pandemic.
Public health strategies to manage epidemics include anticipation and early detection of emerging diseases (aided by rapid development of diagnostics), containment (using strategies to test, trace and isolate such as widespread testing, contact tracing, quarantining and isolation), control and mitigation of spread (including social distancing), and elimination and eradication. 
An effective public health response requires response coordination, disease surveillance and intervention monitoring, risk communication and health education (including education about personal protective equipment and hand hygiene and containment of “infodemics” of false information), non-pharmaceutical measures to prevent and reduce transmission, and health interventions such as vaccines and other pharmaceuticals that can reduce transmission, curb morbidity and mortality, and mitigate the effects on health systems and other sectors of society [@url:https://www.who.int/emergencies/diseases/managing-epidemics-interactive.pdf].
Currently, this manuscript focuses primarily on contextualizing epidemiological characteristics such as reproduction number and dynamics of transmission that intersect with the fundamental biology of the virus or the development of therapeutic and diagnostic technologies.

#### Biotechnology

##### Diagnostics

Two major concerns within diagnosis include the detection of current infections in individuals with and without symptoms, and the detection of past exposure without an active infection.
In the latter category, identifying whether individuals can develop or have developed sustained immunity is also a major consideration.
The development of high-throughput, affordable methods for detecting active infections and sustained immunity will be critical to understanding and controlling the disease.

- What are approaches that allow us to detect current infection or past exposure for other viruses?
- What is sustained immunity and what are the indicators?

##### Therapeutics

The identification of interventions that can mitigate the effect of the virus on exposed and infected individuals is a significant research priority.
Some possible approaches include the identification of existing pharmaceuticals that reduce the severity of infection, either by reducing the virus' virulence (e.g., antivirals) or managing the most severe symptoms of infection.
Due to the long timeline for the development of novel pharmaceuticals, in most cases, research surrounding possible pharmaceutical interventions focuses on the identification and investigation of existing compounds whose mechanisms may be relevant to COVID-19.
Other foci of current research include the identification of antibodies produced by survivors of COVID-19 and the development of vaccines.
Understanding the mechanisms describing host-virus interactions between humans and SARS-CoV-2 are thus critical to identifying candidate therapeutics.

### Summary
In this review, we seek to consolidate information about efforts to develop strategies for diagnosis and therapeutics as new information is released by the scientific community.
We include information from both traditional peer-reviewed scientific literature and from preprints, which typically have not undergone peer review but have been critically evaluated by the scientists involved in this effort.
The goal of this manuscript is to present preliminary findings within the broader context of COVID-19 research and to identify the broad interpretations of new research, as well as limitations to interpretability.
