## Virology

Examining the strcture and basic functions of the SARS-CoV-2 virus is useful for understanding and predicting its pathogenesis and transmission.
Genomic analysis of the virus provided early information about its phylogenetic position among other coronaviruses.
These viruses are widely studied and their genomes are highly conserved, leading to strong hypotheses early on about how the structure and functions of the virus. 

### Coronaviruses

Coronaviruses (CoVs; order *Nidovirales*, family *Coronaviridae*, subfamily *Orthocoronavirinae*) are identified by their distinctive "crown-like" shape.
These large viruses have spherical virions ranging from 100 to 160 nanometers (nm) in diameter.
The virion is made up of a lipid envelope in which peplomers of 2-3 spike (S) glycoproteins are anchored, creating the crown [@isbn:978-0781760607; @doi:10/dvvg2h].
Coronaviruses are classified into four genera: alpha, beta, delta and gamma coronaviruses.
Among them, alpha and beta coronaviruses infect mammalian species, gamma coronaviruses infect avian species and delta coronaviruses infect both mammalian and avian species [@doi:10.1146/annurev-virology-110615-042301].
The viruses were initially subdivided into these genera based on antigenic relationships of the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins but are now divided by phylogenetic clustering.
<!-- this section may be better as a figure panel + legend-->
Phylogenetic analysis of a PCR amplicon fragment from five patients along with the total virus genome of 29.8 Kb indicates that SARS-CoV-2 is a novel betacoronavirus belonging to the B lineage, also known as sarbecovirus, which also includes the human SARS coronavirus [@doi:10/ggjs7j].

### Genome Structure

All viruses in the *Nidovirales* order share fundamental characteristics: they are enveloped and non-segmented and have positive-sense (ssRNA+) genomes that contain many non-structural genes originating through ribosomal frameshifting [@doi:10.1007/978-1-4939-2438-7_1].
Coronaviruses have some of the largest viral RNA genomes, ranging from 27 kilobases (Kb) to 32 Kb in length.
The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp [@doi:10/ggjr43].
Genome organization is highly conserved within the order [@doi:10.1007/978-1-4939-2438-7_1].
A replicase gene comprises about two-thirds of the genome of coronaviruses.
This polypeptide is translated into 16 non-structural proteins (nsp1-16, except in Gammacoronaviruses, where nsp1 is absent) that form replication machinery used to synthesize viral RNA [@doi:10.1002/jmv.25681].
The remaining third of the genome encodes structural proteins, including the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins.
Additional accessory genes are sometimes present between these two regions, depending on the species or strain.
<!-- TODO: Add a figure showing the structure of the genome.-->
<!-- Also, everything focuses on S! Should we also talk about others?-->
Much attention has been focused on the S protein, which is a critical structure involved in cell entry.
This structure has been visualized using cryo-electron microscopy [@doi:10.1126/science.abb2507].

### Mechanisms of Cell Entry

In SARS-CoV, the most similar hCoV, the replication process is initiated by the viral spike protein: first, the virus binds to a host cell, and then the viral membrane fuses with the endosomal membrane to release the viral genome into the host cytoplasm.
The coronavirus spike protein is structured in three segments: the ectodomain, a transmembrane anchor, and an intracellular tail [@doi:10.1128/JVI.02615-14].
The ectodomain contains two subdomains known as the S1 and S2 subunits, with the S1 (N-terminal) subunit guiding the binding of the virus to a host cell receptor and the S2 (C-terminal) subunit guiding the fusion process [@doi:10.1128/JVI.02615-14].
<!-- Is it worth going into more detail about siganl peptides and secretory pathways? Not sure whether this is relevant information.-->
The ectodomain forms the crown-like structures on the viral membrane, with the S1 domain forming the head of the crown and containing the Receptor Binding Motif (RBM) that binds a host receptor, while the S2 domain forms the stalk that supports the head [@doi:10.1038/nrmicro2090].
Coronaviruses bind to a range of host receptors [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009], with binding conserved only at the genus level [@doi:10.1146/annurev-virology-110615-042301].
Betacoronaviruses are known to bind to CEACAM1, Neu 5,9 Ac2, and Angiotensin-Converting Enzyme 2 (ACE2) [@doi:10.3390/v4061011].
Similar to SARS-CoV, SARS-CoV-2 exhibits redundancy in which host proteases it can use to cleave the S protein [@doi:10.1016/j.cell.2020.02.052].
Specifically, both transmembrane protease serine protease-2 (TMPRSS2) and cathepsins B/L have been shown to mediate SARS-CoV-2 S protein proteolytic priming, and small molecule inhibition of these enzymes fully inhibited viral entry _in vitro_ [@doi:10.1016/j.cell.2020.02.052; @doi:10.1038/s41467-020-15562-9].

After the virus binds to a host cell receptor, the viral and plasma membranes must then fuse.
Many coronaviruses are cleaved at the S1-S2 boundary and remain non-covalently bound until fusion to the host cell is achieved, with the S1 subunit stabilizing the S2 subunit during the membrane fusion process [@doi:10.1016/j.cell.2020.02.058].
This priming of the S protein is induced by the binding of cellular proteases [@doi:10/cf8chz; @doi:10.1016/j.antiviral.2020.104792].
A second cleavage site within S2 is thought to activate _S_ for fusion by inducing conformational change [@doi:10.1016/j.cell.2020.02.058].
Electron microscopy suggests that in some coronaviruses, including SARS-CoV and MERS-CoV, a six-helix bundle separates the two subunits in the postfusion conformation, and the unusual length of this bundle facilitates membrane fusion through the release of additional energy [@doi:10.1146/annurev-virology-110615-042301].
However, research in SARS-CoV suggests that, in addition to fusion with the plasma membrane, viruses may also be taken up by cells through endocytosis [@doi:10.1038/cr.2008.15].
<!-- This is very interesting and seems like it could be relevant to virulence (?), but I don't know enough cell biology to expand on it. Another reference that seems useful is doi.org/10.1146/annurev-biochem-060208-104626-->
Once the virus enters a host cell, the replicase gene is translated and assembled into the viral replicase complex, which can synthesize dsRNA genome from the genomic ssRNA(+).
Finally, the dsRNA genome is transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes [@doi:10.1007/978-1-4939-2438-7_1; @url:https://viralzone.expasy.org/30?outline=all_by_species].

### Host-CoV Dynamics

Most coronaviruses are considered zoonotic viruses with little to no transmission observed in humans.
However, coronaviruses include human coronavirus (HCoV) strains associated with multiple respiratory diseases of varying severity, ranging from common cold to severe pneumonia, with severe symptoms mostly observed in immunocompromised individuals [@doi:10.1086/377612].
Approximately one-third of common cold infections in humans is attributable to four out of six previously known human coronaviruses (HCoV-229E, HCoV-NL63, HCoV-OC43 and HCoV-HKU1) that are globally circulating in the human population [@doi:10.1038/s41579-018-0118-9;@doi:10.3390/diseases4030026].
In the past two decades, however, highly pathogenic human coronaviruses have been identified, including the severe acute respiratory syndrome coronavirus 1 (SARS-CoV-1) and the Middle East respiratory syndrome coronavirus (MERS-CoV) although both infections were largely confined to specific geographic regions [@doi:10.1038/nrmicro.2016.81;@doi:10.1038/s41579-018-0118-9; @doi:10.1016/bs.aivir.2018.01.001].

### Evolution and Virulence

Comparative genomic analysis reveals that several regions of the coronavirus genome are likely critical to virulence. 
The S1 domain of the spike protein, which contains the RBM, evolves more rapidly than _S_'s S2 domain [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009].
However, even within the S1 domain, some regions are more conserved than others, with the receptors in S1's N-terminal domain (S1-NTD) evolving more rapidly than those in its C-terminal domain (S1-CTD) [@doi:10.1016/j.jmb.2020.04.009].
Both S1-NTD and S1-CTD are involved in receptor binding and can function as receptor binding domains (RBDs) to bind proteins and sugars [@doi:10.3390/v4061011], but RBDs in the S1-NTD typically bind to sugars, while those in the S1-CTD recognize protein receptors [@doi:10.1146/annurev-virology-110615-042301].
Viral receptors show higher affinity with protein receptors than sugar receptors [@doi:10.1146/annurev-virology-110615-042301], which suggests that positive selection on or relaxed conservation of the S1-NTD preferentially might reduce the risk of a deleterious mutation that would prevent binding.
The SARS-CoV-2 S protein also contains a RRAR furin recognition site at the S1/S2 junction [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507], setting it apart from bat coronavirus RaTG13, with which it shares 96% genome sequence identity, and SARS-CoV [@doi:10.1038/s41586-020-2012-7].
Such furin cleavage sites are commonly found in highly virulent influenza viruses, and as such may contribute to the heightened pathogenicity of SARS-CoV-2 [@doi:10/bvgh5b; @doi:10.1006/viro.1999.9716].
Cell entry by the virus is a critical component to pathogenesis and therefore an important process to understand when examining possible therapeutics.

Evolution in SARS-CoV-2 has also been observed over a short timescale.
After zoonotic transfer, SARS-CoV-2 continued evolving in the human population [@doi:10.1186/s12967-020-02344-6].
The SARS-CoV-2 mutation rate is moderate compared to other RNA viruses [@doi:10.1016/j.meegid.2020.104351], and that restricts SARS-CoV-2 evolution pace.
Nevertheless, genomic data show some significant statistical evidence of ongoing evolution.
There are two known variants of the spike protein that differ by a single amino acid at position 614 (G614 and D614), and there is evidence that G614 has already become dominant over D614 by June 2020 [@doi:10.1016/j.cell.2020.06.043].
While there is a hypothesis that this genomic change increased the SARS-CoV-2 infectivity and virulence, there is no data yet to support or discard this hypothesis reliably [@doi:10.1016/j.cell.2020.06.040].
One study [@doi:10.1016/j.meegid.2020.104351] has reported more than 198 recurrent mutations using a dataset of 7666 curated sequences showing the dynamic of SARS-CoV-2 evolution.
Much uncertainty remains about SARS-CoV-2's evolutionary trends though it is evident that SARS-CoV-2 exhibits moderate potential for further evolution.

