## Virology

In January 2020, the release of the genomic sequence of the virus causing COVID-19, extracted from patient BALF, provided early insights into its phylogenetic position and genomic structure.
As discussed above, the virus clusters with known coronaviruses (order *Nidovirales*, family *Coronaviridae*, subfamily *Orthocoronavirinae*).
Coronaviruses are large viruses that can be identified by the distinctive "crown-like" shape.
Their spherical virions are made from lipid envelopes ranging from 100 to 160 nanometers (nm) in which peplomers of 2-3 spike (S) glycoproteins are anchored, creating the crown [@isbn:978-0781760607; @doi:10/dvvg2h].
These spikes, which are critical to viral pathogenesis and host immune response, have been visualized using cryo-electron microscopy [@doi:10.1126/science.abb2507].
Because many mechanisms of pathogenicity are conserved within the order, this phylogenetic analysis provided an important basis for forming hypotheses about how the virus interacts with hosts.
Additionally, the genomes of coronaviruses are highly conserved, meaning that the relationship between the SARS-CoV-2 genome and its pathogenesis can be inferred from prior research in other viral species.
Phylogenetic analysis of the coronavirus clade indicates four major subclades, corresponding to genera: the alpha, beta, delta and gamma coronaviruses.
Among them, alpha and beta coronaviruses infect mammalian species, gamma coronaviruses infect avian species, and delta coronaviruses infect both mammalian and avian species [@doi:10.1146/annurev-virology-110615-042301].
<!-- TO DO: this section may be better as a figure panel + legend-->
Phylogenetic analysis of a PCR amplicon fragment from five patients along with the full genomic sequence revealed SARS-CoV-2 to be a novel betacoronavirus belonging to the B lineage, also known as sarbecovirus, which also includes the coronavirus SARS-CoV that causes SARS in humans [@doi:10/ggjs7j].
Because viral structure and mechanisms of pathogenicity are highly conserved within the order, this phylogenetic analysis provides a basis for forming hypotheses about how the virus interacts with hosts.

### Genome Structure

The genomes of viruses in the *Nidovirales* order share fundamental characteristics.
They are non-segmented, which means the viral genome is contained in a single capsid, and are enveloped, which means that the genome and capsid are encased by a lipid bilayer.
Coronaviruses have large positive-sense RNA (ssRNA+) genomes ranging from 27 kilobases (Kb) to 32 Kb in length [@doi:10.1007/978-1-4939-2438-7_1; @doi:10/ggjr43].
The SARS-CoV-2 genome lies in the middle of this range at 29,903 bp [@doi:10/ggjr43].
Genome organization is highly conserved within the order [@doi:10.1007/978-1-4939-2438-7_1].
There are three major regions: one containing the replicase gene and one containing the genes encoding structural proteins [@doi:10.1007/978-1-4939-2438-7_1]. 
The replicase gene comprises about two-thirds of the genome of coronaviruses and consists of two open reading frames that are translated with ribosomal frameshifting [@doi:10.1007/978-1-4939-2438-7_1].
This polypeptide is then translated into 16 non-structural proteins (nsp1-16, except in Gammacoronaviruses, where nsp1 is absent) that form replication machinery used to synthesize viral RNA [@doi:10.1002/jmv.25681].
The remaining third of the genome encodes structural proteins, including the spike (S), membrane (M), envelope (E), and nucleocapsid (N) proteins.
Additional accessory genes are sometimes present between these two regions, depending on the species or strain.
<!-- TODO: Add a figure showing the structure of the genome.-->
Much attention has been focused on the S protein, which is a critical structure involved in cell entry.

### Pathogenic Mechanisms of Coronaviruses

While, like most viruses, it is possible that SARS-CoV and therefore SARS-CoV-2 can enter cells through endocytosis, coronaviruses are able to target cells for entry through fusion with the plasma membrane, viruses may also be taken up by cells through endocytosis [@doi:10.1038/cr.2008.15; @doi.org:10.1146/annurev-biochem-060208-104626].
This process is conserved among coronaviruses and is closely associated with the content of their genomes.
Cell entry proceeds in three steps: binding, cleavage, and fusion.
First, the viral spike protein binds to a host cell via a recognized receptor.
Coronaviruses can bind to a range of host receptors [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009], with binding conserved only at the genus level [@doi:10.1146/annurev-virology-110615-042301].
In the betacoronavirus genome, to which SARS-CoV-2 belongs, viruses are known to bind to CEACAM1, Neu 5,9 Ac2, and Angiotensin-Converting Enzyme 2 (ACE2) [@doi:10.3390/v4061011].
SARS-CoV-2 has a high affinity for the human ACE2 receptor, which is expressed in the vascular epithelium, other epithelial cells, and cardiovascular and renal tissues [@doi:10.1007/s00134-020-05985-9; @doi:10.1007/978-0-387-33012-9_85].
The binding process is guided by the molecular structure of the spike protein, which is structured in three segments: an ectodomain, a transmembrane anchor, and an intracellular tail [@doi:10.1128/JVI.02615-14].
The ectodomain forms the crown-like structures on the viral membrane and contains two subdomains known as the S1 and S2 subunits [@doi:10.1038/nrmicro2090].
The S1 (N-terminal) domain forms the head of the crown and contains the receptor binding motif (RBM), and the S2 (C-terminal) domain forms the stalk that supports the head [@doi:10.1038/nrmicro2090].
The S1 subunit guides the binding of the virus to a host cell receptor, and the S2 subunit guides the fusion process [@doi:10.1128/JVI.02615-14].

After the binding of the S1 subunit to a receptor, the spike protein is often cleaved at the S1-S2 boundary by a host protease [@doi:10.1016/j.cell.2020.02.058]; @doi:10/cf8chz; @doi:10.1016/j.antiviral.2020.104792.
Similar to SARS-CoV, SARS-CoV-2 exhibits redundancy in which host proteases it can use to cleave the S protein [@doi:10.1016/j.cell.2020.02.052].
Specifically, both transmembrane protease serine protease-2 (TMPRSS2) and cathepsins B/L have been shown to mediate SARS-CoV-2 S protein proteolytic priming, and small molecule inhibition of these enzymes fully inhibited viral entry _in vitro_ [@doi:10.1016/j.cell.2020.02.052; @doi:10.1038/s41467-020-15562-9].
Proteolytic priming prepares the S protein for fusion [@doi:10/cf8chz; @doi:10.1016/j.antiviral.2020.104792].
The two subunits remain bound by Van der Waal's forces, with the S1 subunit stabilizing the S2 subunit during the membrane fusion process [@doi:10.1016/j.cell.2020.02.058].
Electron microscopy suggests that in some coronaviruses, including SARS-CoV and MERS-CoV, a six-helix bundle separates the two subunits in the postfusion conformation, and the unusual length of this bundle facilitates membrane fusion through the release of additional energy [@doi:10.1146/annurev-virology-110615-042301].
Cleavage at a second site within S2 by these same proteases activates _S_ for fusion by inducing conformational changes [@doi:10.1016/j.cell.2020.02.058].
The viral membrane can then fuse with the endosomal membrane to release the viral genome into the host cytoplasm.
Once the virus enters a host cell, the replicase gene is translated and assembled into the viral replicase complex.
This complex then synthesizes the dsRNA genome from the genomic ssRNA(+).
The dsRNA genome is transcribed and replicated to create viral mRNAs and new ssRNA(+) genomes [@doi:10.1007/978-1-4939-2438-7_1; @url:https://viralzone.expasy.org/30?outline=all_by_species].
From there, the virus can spread in other cells.
Thus, the genome of SARS-CoV-2 is likely to provide insight into the pathogenic behavior of the virus.

### Evolution and Virulence

Comparative genomic analysis reveals that several regions of the coronavirus genome are likely critical to virulence. 
The S1 domain of the spike protein, which contains the RBM, evolves more rapidly than _S_'s S2 domain [@doi:10.3390/v4061011; @doi:10.1016/j.jmb.2020.04.009].
However, even within the S1 domain, some regions are more conserved than others, with the receptors in S1's N-terminal domain (S1-NTD) evolving more rapidly than those in its C-terminal domain (S1-CTD) [@doi:10.1016/j.jmb.2020.04.009].
Both S1-NTD and S1-CTD are involved in receptor binding and can function as receptor binding domains (RBDs) to bind proteins and sugars [@doi:10.3390/v4061011], but RBDs in the S1-NTD typically bind to sugars, while those in the S1-CTD recognize protein receptors [@doi:10.1146/annurev-virology-110615-042301].
Viral receptors show higher affinity with protein receptors than sugar receptors [@doi:10.1146/annurev-virology-110615-042301], which suggests that positive selection on or relaxed conservation of the S1-NTD preferentially might reduce the risk of a deleterious mutation that would prevent binding.
The SARS-CoV-2 S protein also contains a RRAR furin recognition site at the S1/S2 junction [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507], setting it apart from bat coronavirus RaTG13, with which it shares 96% genome sequence identity, and SARS-CoV [@doi:10.1038/s41586-020-2012-7].
Such furin cleavage sites are commonly found in highly virulent influenza viruses, and as such may contribute to the heightened pathogenicity of SARS-CoV-2 [@doi:10/bvgh5b; @doi:10.1006/viro.1999.9716].
Effective cell entry is a critical component to pathogenesis and therefore an important process to understand when examining possible therapeutics.

Evolution in SARS-CoV-2 has also been observed over a short timescale.
After zoonotic transfer, SARS-CoV-2 continued evolving in the human population [@doi:10.1186/s12967-020-02344-6].
The SARS-CoV-2 mutation rate is moderate compared to other RNA viruses [@doi:10.1016/j.meegid.2020.104351], which likely restricts the pace of evolution in SARS-CoV-2.
Nevertheless, genomic analyses have yielded statistical evidence of ongoing evolution.
There are two known variants of the spike protein that differ by a single amino acid at position 614 (G614 and D614), and there is evidence that G614 had become more prevalent than D614 by June 2020 [@doi:10.1016/j.cell.2020.06.043].
While there is a hypothesis that this genomic change increased the SARS-CoV-2 infectivity and virulence, this hypothesis has not yet been tested due to a lack of data [@doi:10.1016/j.cell.2020.06.040].
One study [@doi:10.1016/j.meegid.2020.104351] identified 198 recurrent mutations in a dataset of 7666 curated sequences.
This patterns of convergent evolution suggests that some of the sites may indicate that they confer an adaptive advantage.
While it is evident that SARS-CoV-2 exhibits moderate potential for ongoing or future evolution, the relationship between mutations and pathogenicity is not yet known.
Additional data is needed in order to understand patterns of evolutionary change and whether they are likely to affect virulence.

