## Pathogenesis

Pathogenesis is made up of three major components: entry, replication, and spread [@isbn:0-9631172-1-1 {chap. 45}].
The basic mechanisms of cell entry are known to be similar among coronaviruses and are discussed above.
HCoV are not common, but  combining existing research in SARS-CoV and new research in SARS-CoV-2 led to early hypotheses about how the SARS-CoV-2 virus was likely to infect human cells. 
These insights led to predictions about which tissues, organs, and systems would be most susceptible to SARS-CoV-2 infection.
The current pandemic of COVID-19, caused by the severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), represents an acute global health crisis with symptoms for reported cases ranging from mild to severe or fatal [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html] and including outcomes such as acute respiratory distress, acute lung injury and other pulmonary complications.

### Cell Susceptibility

ACE2 and Transmembrane Serine Protease 2 (TMPRSS2) have been identified as a prime receptor and a critical protease, respectively, facilitating SARS-CoV/CoV-2 entry into a target cell [@doi:10.1038/nature02145; @doi:10.1126/science.abb2507; @doi:10.1128/JVI.01542-10; @doi:10.1016/j.cell.2020.02.052; @doi:10.1128/JVI.02232-10].
This finding has led to a hypothesized role for ACE2 and TMPRSS2 expression in determining which cells, tissues and organs are most likely to be infected by SARS-CoV-2.
The ACE2 receptor is expressed in numerous organs, such as the heart, kidney, and intestine, but most prominently in alveolar epithelial cells, which is expected to contribute to the virus' association with lung pathology [@doi:10.1101/2020.02.24.963348; @doi:10.1007/s00134-020-05985-9].
Clinical investigations of COVID-19 patients have detected SARS-CoV-2 transcripts in bronchoalveolar lavage fluid (93% of specimens), sputum (72%), nasal swabs (63%), fibrobronchoscopy brush biopsies (46%), pharyngeal swabs (32%), feces (29%) and blood (1%) [@doi:10.1001/jama.2020.3786].
Two studies reported that SARS-CoV-2 could not be detected in the urine specimens [@doi:10.1001/jama.2020.3786; @doi:10.1001/jama.2020.3204]; however, a third study identified four urine samples (out of 58) that were positive for SARS-CoV-2 nucleic acids [@doi:10.1097/CM9.0000000000000774].
Respiratory failure remains the leading cause of death for COVID-19 patients [@doi:10.1111/his.14134].
Besides major pulmonary damage, SARS-CoV-2 infection can damage many other organ systems including the heart [@doi:10.1001/jamacardio.2020.0950], kidney [@doi:10.1038/s41585-020-0319-7; @doi:10.1186/s13054-020-02872-z], liver [@doi:10/ggpx6s], and gastrointestinal tract [@doi:10.1053/j.gastro.2020.02.055; @doi:10.1111/1751-2980.12851].
As it becomes clearer that SARS-CoV-2 infection can damage multiple organs, the scientific community is pursuing multiple avenues of investigation in order to build a consensus about how the virus affects the human body.

To quickly associate the clinical outcomes with SARS-CoV-2 infection, researchers are taking advantage of “omics” technologies to profile the expression of coronavirus entry factors across the tissues. <!-- TODO: refs? if it stays here -->
Viral progression may be enhanced by active upregulation of ACE2 on cell surfaces following or during a response to infection.
In several preliminary assays and an analysis of previous microarray data, Wang et al. reported that ACE2 is significantly upregulated following infection by other coronaviruses, including SARS-CoV and MERS-CoV, as well as viruses such as rhinovirus and influenza virus [@doi:10.1101/2020.02.24.963348].
Additionally, direct stimulation with inflammatory cytokines such as type I interferons resulted in the upregulation of ACE2, with treated groups showing 4-fold higher ACE2 expression than control groups at 18 hours post-treatment [@doi:10.1101/2020.02.24.963348].
Though whether SARS-CoV-2 infection facilitates positive regulation of its own transmission between host cells is still unclear, the host immune response itself likely plays a key role in mediating infection-associated pathologies.
One severe example includes reports of cytokine storm-like responses in patients with particularly severe infections, in which the overproduction of inflammatory cytokines leads to systemic inflammation and potentially multi-organ failure, which may very well accelerate the spread of virus in the host [@doi:10.1101/2020.02.24.963348; @doi:10/ggnzmc].
[@doi:10.1016/j.immuni.2020.06.017] describe the concept of the cytokine storm in detail, although [@doi:10.1001/jamainternmed.2020.3313] claim that the widespread use of the term can be misleading.

### Pathogenicity of SARS-CoV-2 in Humans

#### Virulence

Like that of SARS-CoV, SARS-CoV-2 entry into host cells is mediated by the interaction between the viral spike glycoprotein (S) and human angiotensin-converting enzyme 2 (ACE2) [@doi:10.1126/science.1116480; @doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2762; @doi:10.1038/s41586-020-2179-y; @doi:10.1101/2020.02.19.956235; @doi:10.1016/j.cell.2020.02.052; @doi:10.1016/j.cell.2020.03.045; @doi:10.1128/jvi.00127-20].
Differences between the two viruses in their interactions between the S protein and hACE2 may also partially account for the increased transmissibility seen in the current COVID-19 pandemic.
Recent studies have reported conflicting binding constants for the S protein-hACE2 interaction, though they agree that SARS-CoV-2 S protein binds with equal if not greater affinity than does SARS-CoV S protein [@doi:10.1016/j.cell.2020.02.058; @doi:10.1101/2020.02.19.956235; @doi:10.1126/science.abb2507].
The C-terminal domain of the SARS-CoV-2 S protein in particular was identified as the key region of the virus that interacts with the human ACE2 (hACE2) receptor, and the crystal structure of the C-terminal domain of SARS-CoV-2 S protein in complex with human ACE2 reveals stronger interaction and higher affinity for receptor binding than SARS-CoV [@doi:10.1016/j.cell.2020.03.045].
Among the 14 key binding residues identified in the SARS-CoV S protein, 8 are conserved in SARS-CoV-2 and the remaining 6 are semi-conservatively substituted, potentially explaining variation in binding affinity [@doi:10.1016/j.cell.2020.02.058; @doi:10.1101/2020.02.19.956235].
Recent crystal structures have shown that the receptor-binding domain (RBD) of SARS-CoV-2 S protein, like that of other coronaviruses, undergoes stochastic hinge-like movement that flips it from a “closed” conformation, in which key binding residues are hidden at the interface between protomers, to an “open” one [@doi:10.1016/j.cell.2020.02.058; @doi:10.1126/science.abb2507].
Because the RBD plays such a critical role in viral entry, blocking its interaction with ACE2 represents a promising therapeutic approach.
Nevertheless, despite the high structural homology between SARS-CoV-2 RBD and that from SARS-CoV, monoclonal antibodies targeting SARS-CoV-RBD failed to bind SARS-CoV-2-RBD [@doi:10.1126/science.abb2507].
Promisingly though, sera from convalescent SARS patients inhibited SARS-CoV-2 viral entry _in vitro_, albeit with lower efficiency than it inhibited SARS-CoV [@doi:10.1016/j.cell.2020.02.052].

#### Evasion of Immune Response

Research in other coronaviruses that affect humans provides some indication of how SARS-CoV-2 infection proceeds in spite of the human immune response.
By infecting the epithelium, viruses such as SARS-CoV are known to bypass the physical barriers such as skin and mucus that comprise the immune systems' first line of defense [@doi:10.1038/mi.2015.127].
Once the virus infiltrates host cells, it is adept at evading detection. 
CD163+ and CD68+ macrophage cells especially are crucial for the establishment of SARS-CoV in the body [@doi:10.1038/mi.2015.127].
These cells most likely serve as viral reservoirs that help shield SARS-CoV from the innate immune response.
According to a study on the viral dissemination of SARS-CoV in Chinese macaques, viral RNA could be detected in some monocytes throughout the process of differentiation into dendritic cells [@doi:10.1038/mi.2015.127].
This lack of active viral replication allows SARS-CoV to escape innate immunity because reduced levels of detectable viral RNA allow the virus to avoid both natural killer (NK) cells and Toll-like receptors [@doi:10.1038/mi.2015.127].
Even during replication, SARS-CoV is able to mask its dsRNA from detection by the immune system.
Although double-stranded RNA is a pathogen-associated molecular pattern (PAMP) that would typically initiate a response from the innate immune system [@doi:10.3389/fimmu.2018.00829], _in vitro_ analysis of nidoviruses including SARS-CoV suggests that these viruses can induce the development of double-membrane vesicles that protect the dsRNA signature from being detected by the host immune system [@doi:10.1128/jvi.02501-05].
This protective envelope can therefore insulate these coronaviruses from the innate immune response's detection mechanism [@doi:10.1016/j.jpha.2020.03.001].

HCoV are also known to interfere with the host immune response, rather than just evade it.
For example, the virulence of SARS-CoV is increased by nonstructural protein 1 (Nsp1), which can suppress host gene expression.
Nsp1 accomplishes this by stalling mRNA translation and inducing endonucleolytic cleavage and mRNA degradation [@doi:10.1126/science.abc8665].
SARS-CoV also evades the immune response by interfering with type I IFN induction signaling, which is a mechanism that leads to cellular resistance to viral infections.
SARS-CoV employs methods such as ubiquitination and degradation of RNA sensor adaptor molecules MAVS and TRAF3/6 [@doi:10.12932/AP-200220-0772].
Also, MERS-CoV downregulates antigen presentation via MHC class I and MHC class II, which leads to a reduction in T cell activation [@doi:10.12932/AP-200220-0772].
These evasion mechanisms, in turn, can lead to systemic infection.
Coronaviruses such as SARS-CoV are also able to evade the humoral immune response through other mechanisms, such as inhibiting certain cytokine pathways or down-regulating antigen presentation by the cells [@doi:10.1128/jvi.02501-05].

<!--
#### Acquired Immunity
TODO: Move and/or update this section
In the process of mounting a response to a viral pathogen, the immune system will produce antibodies specific to the pathogen.
Understanding the acquisition and retention of antibodies is important to both the diagnosis of prior infections that are no longer active and to the development of vaccines.
Two of the most common immunoglobulin classes are immunoglobulin M (IgM), which are the first antibodies produced in response to an infection, and immunoglobulin G (IgG), which are the most common antibodies found in the blood.
Following SARS infection, IgM antibodies were first detected in most patients during the second week following infection and remained detectable until the twelfth week, while IgG antibodies were first detected in the second or third week and remained at close to their peak concentration through the thirteen weeks of the study [@doi:10.1056/NEJM200307313490520].
A two-year longitudinal study following convalesced SARS patients with a mean age of 29 found that IgG antibodies were detectable in all 56 patients surveyed for at least 16 months, and remained detectable in all but 4 (11.8%) of patients through the full two-year study period [@doi:10.1086/500469].
-->

#### Cytokine Storm

In progress
<!--
Text is not fluid as it has been moved from several different places in the manuscript

Disease symptoms can be caused by a pathogen or by the reaction of the immune system to the pathogen.
The host immune system can defend against infection but can also sometimes cause too much damage to the host [@isbn:9780815332183; @isbn:9780071283663].
There is a review on interferon-mediated innate antiviral responses initiated by the virus-infected cells themselves, which can sometimes have inflammatory and other systemic effects [@doi:10.1016/j.chom.2020.05.008].
Also, there is a review on the innate immune system response with a focus on innate immune cells and responses other than the interferon-mediated responses; the review also approaches the question of how the innate immune response can confer protection or induce pathogenesis through a dysregulated response [@doi:10.1016/j.chom.2020.05.009].
There are also acquired immunity responses to SARS-CoV-2.
There is a review on the levels of different SARS-CoV-2-specific antibodies detected over the course of an infection [@doi:10.3389/fimmu.2020.00879].
A review of T-cell mediated responses to respiratory coronaviruses notes a large reduction in T-cells associated with the acute phase of SARS (from SARS-CoV-1) [@doi:10.1007/s12026-014-8534-z].
So far, some articles and reviews on SARS-CoV-2 mention evidence for reduction in the number of CD4 and CD19 T-cells as well as NK-cells [@doi:10.1016/j.chom.2020.04.009; @doi:10.1016/j.chom.2020.05.009].
As a brief immunology background primer for vaccines, which we discuss later, we note that adaptive immunity also has mechanisms for long-term immunity: during a first exposure to a pathogen innate and also later acquired immune responses operate; at the same time, the acquired immune system forms memory B cells and memory T cells based on this experience so that on a possible future second exposure to the same pathogen an earlier/faster and stronger acquired immune response is generated which largely prevents or greatly reduces infection.
Vaccines are in general attenuated or inactivated pathogens or antigen parts of pathogens which are meant to generate long-term memory of a pathogen so that if a person is exposed to the same pathogen in the future they will have a strong immunity memory response; we will describe specific research into vaccines in detail later.
Other immune system features and pathogen-immune system interactions which are related to particular diagnostics and therapeutics are described within the Diagnostics and Therapeutics sections.

IL-6 promotes the differentiation of activated B cells into immunoglobulin-producing plasma cells [@pmid:9052836] and acts as a growth factor for hybridoma and myeloma cells [@pmid:8431556; @pmid:2104241].
In addition, IL-6 also induces the differentiation of naïve CD4+ T cells into effector T-cell subsets [@doi:10.1158/2326-6066.CIR-14-0022].
In this way interleukins regulate both the pro- and anti-inflammatory responses.
<!-- Note: make sure these terms are defined!-->
In a healthy situation the lung respiratory epithelium together with alveolar macrophages limits the activation of the immune system, ensuring homeostasis.
The introduction of the S-protein from SARS-CoV to mouse macrophages was found to increase production of IL-6 and TNF-α [@doi:10.1016/j.virusres.2007.02.007], and deceased SARS-CoV patients were found to have intermediate levels of IL-6, IL-1𝛽, and TNF-α expressed in a number of ACE2-expressing cell types sampled from the lung and bronchial tissues during autopsy [@doi:10.1002/path.2067].
However, other reports found the severe respiratory condition ARDS to be associated with elevated concentrations of IL-6 in BALF, but that concentrations of Tumor Necrosis Factor α (TNF-α) and IL-1𝛽 decreased with the onset of ARDS [@doi:10.1164/ajrccm.164.10.2104013].
<!--Note: Just to clarify, is the point here that SARS patients have intermediate levels of IL-6 but other respiratory conditions, like ARDS have higher levels of IL-6? As a non-immunologist, my high level understanding of what is written is: IL-6 plays a role in regulating the immune response to infection. And different IL-6 levels are found depending on the disease. This IL-6 cytokine causes a large immune response that damages cells and is fatal. - @ajlee21 -->
These cytokines enhance the pro-inflammatory reaction by increase acute-phase signaling, trafficking of immune cells to the site of primary infection, epithelial cell activation, and secondary cytokine production.
The acute phase response to infection results in the heavy damage of the endothelium of blood vessels, which disrupts the balance between the pro- and anti-inflammatory response [@doi:10.1164/ajrccm.164.10.2104013].
Thus, the holes generated allow not just for the passage of neutrophils, macrophages and lymphocytes to the site of the infection but also the accumulation of liquids into the lungs, which is the ultimate cause of the death in ARDS and Severe Acute Respiratory Syndrome (SARS) [@doi:10.1007/s00281-017-0629-x], also caused by the new coronavirus.
<!-- Suggestion: define distinction between Cytokine Release Syndrome and Cytokine Storm Syndrome-->
One severe example includes reports of cytokine storm-like responses in patients with particularly severe infections, in which the overproduction of inflammatory cytokines leads to systemic inflammation and potentially multi-organ failure, which may very well accelerate the spread of virus in the host [@doi:10.1101/2020.02.24.963348; @doi:10/ggnzmc].
[@doi:10.1016/j.immuni.2020.06.017] describe the concept of the cytokine storm in detail, although [@doi:10.1001/jamainternmed.2020.3313] claim that the widespread use of the term can be misleading.
-->

### Clinical Presentation of COVID-19

<!-- TODO: there is more literature on this now - we might want to go back and see if we still like this -->
A great diversity of symptom profiles has been observed for COVID-19, although a large study from Wuhan, China suggests fever and cough as the two most common symptoms on admission [@doi:10.1056/NEJMoa2002032].
One early retrospective study in China described the clinical presentations of patients infected with SARS-CoV-2 as including lower respiratory tract infection with fever, dry cough, and dyspnea [@doi:10/ggnxb3].
This study [@doi:10/ggnxb3] noted that upper respiratory tract symptoms were less common, which suggests that the virus targets cells located in the lower respiratory tract.
However, data from the New York City region [@doi:10.1001/jama.2020.6775; @doi:10.1056/NEJMc2010419] showed variable rates of fever as a presenting symptom, suggesting that symptoms may not be consistent across samples.
These differences are present when comparing both between institutions in similar locations and between different regions experiencing COVID-19 outbreaks, leading to conflicting reports of the frequency of fever as a presenting symptom for patients upon hospital admission.
For example, even within New York City, one study [@doi:10.1001/jama.2020.6775] identified low oxygen saturation (<90% without the use of supplemental oxygen or ventilation support) in a significant percentage of patients on presentation, while another study [@doi:10.1056/NEJMc2010419] reported cough, fever, and dyspnea as the most common presenting symptoms.

Patients may also experience loss of smell, myalgias (muscle aches), fatigue, or headache.

The variability of both which symptoms present and their severity makes it difficult for public health agencies to provide clear recommendations for citizens regarding what symptoms indicate SARS-CoV-2 infection and should prompt isolation.

For example, accumulating evidence suggests that gastrointestinal symptoms presentations occur [@doi:10.15585/mmwr.mm6928a2].
The CDC has updated its list of symptoms suggestive of COVID-19 to include nausea and vomiting, as well congestion and runny nose [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html].
A recent preprint using data from an app-based survey of 500,000 individuals in the US found that among those tested for SARS-CoV-2, a loss of taste or smell, fever, and a cough were significant predictors of a positive test result [@doi:10.1101/2020.06.09.20126813].
It is important to note that in this study, the predictive value of symptoms may be underestimated if they are not specific to COVID-19 because the outcome measured was a positive, as opposed to a negative, COVID-19 test result.
At the time the surveys were conducted, due to limits in US testing infrastructure, respondents typically needed to have some symptoms known to be specific to COVID-19 in order to qualify for testing in the first place.

The significant range of individual outcomes observed has led to interest in which factors influence disease severity.
The reported hospitalization rates have been wide-ranging and appear to be influenced by age and location, with several underlying health conditions being disproportionately reported among hospitalized patients [@doi:10.15585/mmwr.mm6915e3].
<!--Should emphasize that the risk goes up a lot with age, with maleness and diabetes/obesity also being risk factors.-->
Among patients who are admitted to the hospital, outcomes have generally been poor, with rates of admission to the intensive care unit (ICU) upwards of 15% in both Wuhan, China and Italy [@doi:10.1056/nejmoa2002032; @doi:10.1001/jama.2020.2648; @doi:10.1001/jama.2020.4031].
Several studies have also investigated which factors are likely to influence COVID-19 outcomes.
For example, a Chinese retrospective study [@doi:10/ggnxb3] found that a higher probability of mortality was associated with older age and higher Sequential Organ Failure Assessment scores, as well as high levels of d-dimer.
Mortality might be associated with other biomarkers measured in blood samples including lactate dehydrogenase and cardiac troponin I, although these analyses may not have been appropriately corrected for multiple testing.
Zhou et al. [@doi:10/ggnxb3] reported that survivors continued to have detectable viral nucleic acids in respiratory specimens for a median of 20 days and a maximum of at least 37 days.
A later study reported radiographic findings such as ground-glass opacity and bilateral patchy shadowing in the lungs of many hospitalized patients, and most COVID-19 patients had lymphocytopenia, meaning they had low levels of lymphocytes (a type of white blood cell) [@doi:10.1056/NEJMoa2002032].

COVID-19 can affect diverse body systems in addition to causing respiratory problems [@doi:10.1038/s41591-020-0968-3].
For example, COVID-19 can lead to acute kidney injury, especially in patients with severe respiratory symptoms or certain preexisting conditions [@doi:10.1016/j.kint.2020.05.006].
It can also cause neurological complications [@doi:10.1016/j.bbi.2020.03.031; @doi:10/d259], potentially including stroke, seizures or meningitis [@doi:10.1002/jmv.26000; @doi:10.1002/ana.25807].

COVID-19 has also been associated with an increased incidence of large vessel stroke, particularly in patients under the age of 40 [@doi:10.1056/NEJMc2009787], and other thrombotic events including pulmonary embolism and deep vein thrombosis [@doi:10.1016/j.thromres.2020.04.013].
The mechanism behind these complications has been suggested to be related to coagulopathy with reports of antiphospholipid antibodies present [@doi:10.1056/NEJMc2007575] and elevated levels of d-dimer and fibrinogen degradation products (FDP) elevated in deceased patients [@doi:10.1111/jth.14768].
Other viral infections have been associated with coagulation defects; notably SARS was also found to lead to disseminated intravascular coagulation (DIC) and was associated with both pulmonary embolism and deep vein thrombosis [@doi:10.1002/jmv.23354].
A wide range of viral infections can affect the coagulation cascade.
The mechanism behind these insults has been suggested to be related to inflammation-induced increases in the von Willebrand factor (VWF) clotting protein, leading to a pro-coagulative state [@doi:10.1002/jmv.23354].
Abnormal clotting (thromboinflammation or coagulopathy) has been increasingly discussed recently as a possible key mechanism in many cases of severe COVID-19, and may be associated with the high d-dimer levels often observed in severe cases [@doi:10/ggvd74; @doi:10.1186/s13054-020-03060-9; @doi:10.1007/s11239-020-02134-3].
This excessive clotting in lung capillaries has been suggested to be related to a dysregulated activation of the complement system, part of the innate immune system [@doi:10.1172/jci.insight.140711; @doi:10.1172/JCI140183].

The presentation of COVID-19 infection can vary greatly among pediatric patients and in some cases manifests in distinct ways from COVID-19 in adults.
[@doi:10.1001/jamapediatrics.2020.1467] reviewed 17 studies on children infected with COVID-19 during the early months of the COVID-19 epidemic in China and one study from Singapore.
Of the more than a thousand cases described, the most commonly reports were for mild symptoms such as fever, dry cough, fatigue, nasal congestion and/or runny nose, while three children were reported to be asymptomatic.
Severe lower respiratory infection was described in only one of the pediatric cases reviewed.
Gastrointestinal symptoms such as vomiting or diarrhea were occasionally reported.
Radiologic findings were not always reported in the case studies reviewed by [@doi:10.1001/jamapediatrics.2020.1467], but when they were mentioned they included bronchial thickening, ground-glass opacities, and/or inflammatory lesions.
Neurological symptoms have also been reported [@doi:10.1001/jamaneurol.2020.2687].

These analyses indicate that generally, most pediatric cases of COVID-19 are not severe.
However, serious complications and, in rare cases, death have occurred [@doi:10.1016/j.eclinm.2020.100433].
Of particular interest, children have occasionally experienced a serious inflammatory syndrome, multisystem inflammatory syndrome in children (MIS-C), after COVID-19 infection.
This syndrome is similar in some respects to Kawasaki disease or to Kawasaki disease shock syndrome [@doi:10.1093/jpids/piaa069; @doi:10.1001/jama.2020.10369; @doi:10.1016/j.ajem.2020.05.117] and is thought to be a distinct clinical manifestation of SARS-CoV-2 due to its distinct cytokine profile and the presence of burr cells in peripheral blood smears [@doi:10.1172/JCI140970].
MIS-C has been associated with heart failure in some cases [@doi:10.1161/CIRCULATIONAHA.120.048360].
[@doi:10/gg4sd6] reported a single case of an adult who appeared to show symptoms similar to MIS-C after exposure to COVID-19, but cautioned against broad conclusions; [@doi:10.1016/j.ajem.2020.06.053] reported another such possible adult case.

### Systems Approaches to Pathogenesis

Systems biology provides a cross-disciplinary analytical platform integrating the different omics (genomics, transcriptomics, proteomics, metabolomics, and other omics approaches), bioinformatics, and computational strategies.
These cutting-edge research approaches have enormous potential to study the complexity of biological systems and human diseases [@doi:10.1016/j.gpb.2012.08.004].
Over the last decade, systems biology approaches have been used widely to study the pathogenesis of diverse types of life-threatening acute and chronic infectious diseases [@doi:10.1038/s41576-020-0212-5].
Omics-based studies also provided meaningful information regarding host immune responses and surrogate protein markers in several viral, bacterial and protozoan infections [@doi:10.1002/prca.201300074].
The complex pathogenesis and clinical manifestations of SARS-CoV-2 infection are not understood adequately yet.
The genome sequence of the pathogen subsequently allowed its phylogenetic characterization and prediction of its protein expression profile, which is crucial for understanding the pathogenesis and virulence of this novel viral infection.

#### Proteomics

<!--Some of the text below might need to be rearranged or  moved to other places in the paper-->
Availability of the genome sequence of SARS-CoV-2 enhances the potential for subsequent proteome-level studies to provide further mechanistic insights into the virus' complex pathogenesis.
Even though no comprehensive proteomic analysis of the pathogen or of patients suffering from its infection has yet been reported, one forthcoming study has demonstrated SARS-CoV-2 infected host cell proteomics using human Caco–2 cells as an infection model [@doi:10.1038/s41586-020-2332-7].
The authors observed SARS-CoV-2 induced alterations in multiple vital physiological pathways, including translation, splicing, carbon metabolism and nucleic acid metabolism in the host cells.
There is a high level of sequence homology between SARS-CoV-2 and SARS-CoV, and sera from convalescent SARS-CoV patients might show some efficacy to cross-neutralize SARS-CoV-2-S-driven entry [@doi:10.1016/j.cell.2020.02.052].
However, despite the high level of sequence homology, certain protein structures might be immunologically distinct, prohibiting effective cross-neutralization across different SARS strains [@doi:10.1101/2020.03.21.990770].
Consequently, earlier proteome-level studies on SARS-CoV can also provide some essential information regarding the new pathogen [@doi:10.1073/pnas.0407992101; @doi:10.1016/j.bbrc.2004.02.074].
Considering the paucity of omics-level big data sets for SARS-CoV-2 up until now, existing data hubs that contain information for other coronaviruses such as UniProt, NCBI Genome Database, The Immune Epitope Database and Analysis Resource (IEDB), and The Virus Pathogen Resource (ViPR) will serve as useful resources for computational and bioinformatics research on SARS-CoV-2.
Using such databases, the systems level reconstruction of the PPI (Protein-Protein Interaction) enabled the generation of hypotheses on the mechanism of action of SARS-CoV-2 and suggested drug targets.
In an initial study [@doi:10.1101/2020.03.22.002386], 26 of the 29 SARS-CoV-2 proteins were cloned and expressed in HEK293T kidney cells, allowing for the identification of 332 high-confidence human proteins that interact with them.
Notably, this study suggested that SARS-CoV-2 interacts with innate immunity pathways.
The ranking of pathogens with respect to their interactome's similarity to SARS-CoV-2 suggested *West Nile Virus*, *Mycobacterium tuberculosis*, and *Human papillomavirus* as the top three hits.
However, given the lung symptoms associated with COVID-19, the *Mycobacterium tuberculosis* host-pathogen interactome could also provide new insights to the mechanism of SARS-CoV-2 infection.
In addition, it was suggested that the envelope protein E could disrupt the host bromodomain-containing proteins, i.e., BRD2 and BRD4, binding to histone.
The Spike protein S could likely intervene in the virus fusion through modulating the GOLGA7-ZDHHC5 acyl-transferase complex to increase palmitoylation.

Another study [@doi:10.1101/2020.03.31.019216], used patient-derived peripheral blood mononuclear cells (PBMCs) to identify 251 host proteins targeted by SARS-CoV-2 and the disruption of more than 200 host proteins following the infection.
The network analysis showed in particular that non-structural proteins 9 and 10 (nsp9 and nsp10) interacted with NKRF that usually represses NFKB.
This finding could explain the exacerbation of the immune response that shape the pathology and the high cytokine levels possibly due to the chemotaxis of neutrophils mediated by IL-8 and IL-6.
Finally, it was suggested [@doi:10.1101/2020.04.09.033522] that protein E of both SARS-CoV and SARS-CoV-2 has a conserved Bcl-2 Homology 3 (BH3)-like motif, that could inhibit anti-apoptosis proteins BCL2 and trigger the apoptosis of T cells.
Several known compounds were suggested to disrupt the host-pathogen protein interactome were suggested mostly through the inhibition of host proteins.

#### Transcriptomics

To quickly associate the clinical outcomes with SARS-CoV-2 infection, researchers are taking advantage of “omics” technologies to profile the expression of coronavirus entry factors across the tissues/cells and also to measure the host transcriptional response after virus infection.
Two studies profiled expression after SARS-CoV-2 infection in multiple human cell lines (Figure {@fig:cell-lines-moi}A) [@doi:10.1016/j.cell.2020.04.026; @doi:10.1101/2020.05.05.079194].
The first study, which is by Blanco-Melo et al., performed an in-depth analysis of the transcriptional response to SARS-CoV-2 in comparison to other respiratory viruses, including MERS-CoV, SARS-CoV, human parainfluenza virus 3 (HPIV3), respiratory syncytial virus (RSV), and influenza A virus (IAV).
They analyzed the responses of three human cell lines: A549 (adenocarcinomic human alveolar basal epithelial cells), Calu-3 (human airway epithelial cells derived from human bronchial submucosal glands), and MRC-5 (human fetal lung fibroblast cells).
As the viral receptor ACE2 has low expression in A549 cells, authors also supplemented A549 cells with adenovirus (AdV)-based vectors expressing mCherry (a fluorescent protein used as a control) or ACE2 (A549-ACE2).
All infections were performed at a high multiplicity of infections (MOI, 2–5), with the exception of A549-ACE2 cells, which were infected with SARS-CoV-2 at both low (0.2 MOI) and high MOI (2 MOI).
Poly(A) bulk RNA sequencing (RNA-seq) of host cells was performed at 24 hours post-infection (hpi) of all the viruses except IAV (9-12 hours).
The authors also measured host transcriptional responses to SARS-CoV-2 in primary normal human bronchial epithelial cells (HBEC or NHBE cells), nasal washes of ferrets as an animal model, and lung samples from two COVID-19 patients.
Additionally, they measured transcriptional responses to IAV in NHBE cells and nasal washes of ferrets.
Differential expression (DE) analysis was then carried out to compare infected cells with control cells that underwent only a mock treatment.
In general, the interferons (IFNs) activate intracellular antimicrobial programs and are the major first line of defense against viruses [@doi:10.1016/j.chom.2020.05.008].
IFNs are a type of cytokines along with chemokines, interleukins (IL), lymphokines and tumor necrosis factor (TNF).
Cytokines can be classified based on the nature of the immune response or based on their specific roles, as reviewed by [@doi:10.1016/j.bbamcr.2014.05.014]).
In the hosts where SARS-CoV-2 was able to replicate efficiently, DE analysis revealed that the transcriptional response was significantly different from the response to all of the other viruses tested.
A unique proinflammatory cytokine signature associated with SARS-CoV-2 was present under both high- and low-MOI conditions.
The cytokines IL-6 and IL1RA were uniquely elevated in response to SARS-CoV-2.
However, A549-ACE2 cells in the low-MOI condition showed no significant IFN-I or IFN-III expression, but did in cells at high MOI.
This finding suggests that IFN induction is MOI dependent.
Similarly, in cells from the NHBE line, ferrets, and COVID-19 patients, they found significant enrichment of chemokine signaling but no significant induction of IFN-I or IFN-III.
Together, these results suggest that in contrast to common respiratory viruses, SARS-CoV-2 induces a limited antiviral state with low IFN-I or IFN-III expression and a moderate IFN-stimulated genes (ISG) response.
However, the low IFN induction in ACE2-expressing A549 cells could be overcome by using a 10-fold increase in SARS-CoV-2 (MOI, 2), suggesting a SARS-CoV-2 antagonist that is rendered ineffective under high-MOI conditions [@doi:10.1016/j.cell.2020.04.026].
This hypothesis was further supported by a recent study [@doi:10.1101/2020.05.11.088179] that showed that the SARS-CoV-2 _ORF3b_ gene suppresses IFNB1 promoter activity (IFN-I induction) more efficiently than the SARS-CoV_ORF3b_ gene, which is considerably longer.

A second study by Wyler et al. [@doi:10.1101/2020.05.05.079194] examined three human cell lines: H1299 (human non-small cell lung carcinoma cell line), Calu-3, and Caco-2 (human epithelial colorectal adenocarcinoma cell line).
They performed a comprehensive analysis of the transcriptional response to SARS-CoV-2 and SARS-CoV at different hpi (4-36 hours) with an MOI of 0.3.
Using poly(A) bulk RNA-seq, the authors found negligible susceptibility of H1299 cells (< 0.08 viral read percentage of total reads) in contrast to Caco-2 and Calu-3 cells (>10% of viral reads), suggesting a role of cell-type specific host factors.
Based on visual inspection (microscopy images) and transcriptional profiling, the authors showed distinct responses associated with different hosts following infection.
In contrast to Caco-2, SARS-CoV-2 infected Calu-3 cells showed signs of impaired growth and cell death at 24 hpi, as well as moderate IFN induction with a strong up-regulation of ISGs.
Interestingly, the response is similar to SARS-CoV-2 infected Calu-3 cells at ~7-fold higher MOI performed by Blanco-Melo et al., suggesting that IFN induction is not MOI dependent in Calu-3 cells, in contrast to reports using A549-ACE2 cells.
The discrepancy could be explained by the observations that Calu-3 cells are highly susceptible to SARS-CoV-2 with rapid virus replication [@doi:10.1038/s41467-020-15562-9], whereas A549 cells are incompatible with SARS-CoV-2 infection [@doi:10.1101/2020.03.02.972935].
We performed a correlation analysis of the host expression response (RNA fold change compared to the mock infection) using the publicly available analyzed data from Blanco-Melo et al. and Wyler et al. (Figure {@fig:cell-lines-moi}B) [@doi:10.1016/j.cell.2020.04.026; @doi:10.1101/2020.05.05.079194].
In general, we found high correlations among the same host irrespective of the study and the virus infected, suggesting that the response is host-specific.
For instance, Calu-3 cells infected with SARS-CoV by Wyler et al. and Calu-3 cells infected with SARS-CoV-2 by Blanco-Melo et al. had a Pearson's correlation coefficient of 0.8 (Figure {@fig:cell-lines-moi}B).
Moreover, COVID-19 patients had the lowest correlations among all the cells used, with the highest value being 0.29.
This discrepancy suggests that the _in vitro_ models used may not necessarily imitate the human response, underscoring the importance of follow-up in additional models.

![**Profiling and correlation of expression response after respiratory viruses infection in various hosts.** (A) Overview of hosts and respiratory viruses used in two studies [@doi:10.1016/j.cell.2020.04.026; @doi:10.1101/2020.05.05.079194]. MOI:Multiplicity of infections; PFU:Plaque-forming unit; a:ACE2-expressing A549 cells; b:Data used from GSE56192. (B) Pearson's correlation of expression response (Log2 fold change) between various hosts. Note that we used the data from bulk RNA-seq produced after the infection of unmodified viruses.](images/cell-lines-moi.jpg){#fig:cell-lines-moi secno=1}

<!-- Start with scRNA-seq studies in healthy tissues (may be put this before Blanco-Melo et al), then summarize scRNA-seq studies in COVID-19 patients] -->
