## Transmission and Susceptibility

### Mechanism of Transmission

Person-to-person viral transmission of a virus can occur through several possible mechanisms.
The primary mechanisms associated with respiratory viruses are contact, droplet, and aerosol transmission [@doi:10.1097/QCO.0000000000000563].
Contact transmission can occur through either direct contact with a contagious person or indirect contact with active viral particles on a contaminated surface [@doi:10.4415/ANN_13_02_03].
This latter mode of transmission is also called fomite transmission [@url:https://www.who.int/news-room/commentaries/detail/transmission-of-sars-cov-2-implications-for-infection-prevention-precautions].
Viral particles can enter the body if they then come in contact with the oral, nasal, eye, or other mucus membranes [@doi:10.4415/ANN_13_02_03].
Droplet transmission occurs when a contagious individual sneezes, coughs, or exhales and produces respiratory droplets that can contain a large number of viral particles [@doi:10.4415/ANN_13_02_03].
Contact with these droplets can occur either through direct exposure to the droplets, such as breathing in droplets produced by a sneeze, or exposure to particles that have settled on a surface [@doi:10.4415/ANN_13_02_03].
Aerosol transmission refers to much smaller particles (less than 5 micrometers) that are also produced by sneezing, coughing, or exhaling [@doi:10.1097/QCO.0000000000000563; @doi:10.4415/ANN_13_02_03].
The small size of these particles allows them to remain suspended over a longer period of time and potentially to be moved by air currents [@doi:10.4415/ANN_13_02_03].
Additionally, viral particles deposited on surfaces via large respiratory droplets can also later be aerosolized [@doi:10.4415/ANN_13_02_03].
Droplet and/or contact transmission are both well-accepted modes of transmission for many viruses associated with common human illness, including influenza and rhinovirus [@doi:10.4415/ANN_13_02_03]. 
The extent to which aerosol transmission contributes to the spread of respiratory viruses is less clear.
In influenza A, for example, viral particles can be detected in aerosols produced by infected individuals, but the extent to which these particles drive the spread of influenza A infection remains under debate [@doi:10.4415/ANN_13_02_03; @doi:10.1097/QCO.0000000000000563; @doi:10.3201/eid1301.061202; @doi:10.1038/s41598-019-38825-y; @doi:10.1093/femsre/fuv039].
Regardless of its role in the spread of influenza A, however, aerosol transmission likely played a role in outbreaks such as the 1918 Spanish Influenza (H1N1) and 2009 "swine flu" (pH1N1) [@doi:10.1093/femsre/fuv039].
Contact, droplet, and aerosol transmission are therefore all worth evaluating when considering possible modes of transmission for a respiratory virus like SARS-CoV-2.

#### Transmission of HCoV

All three of these mechanisms have been identified as contributors to the transmission of HCoV [@doi:10.4415/ANN_13_02_03], including the highly pathogenic coronaviruses SARS-CoV and MERS-CoV [@doi:10.1016/B978-0-12-385885-6.00009-2; @doi:10.1038/nrmicro.2016.81].
SARS transmission is thought to proceed primarily through droplet transmission, but aerosol transmission is also considered possible [@doi:10.4415/ANN_13_02_03], and fomite transmission may have also played an important role in some outbreaks [@doi:10.1371/journal.pone.0181558].
Similarly, the primary mechanism of MERS transmission is thought to be droplets because inter-individual transmission appears to be associated with close interpersonal contact (e.g., household or healthcare settings), but aerosolized particles of the MERS virus have been reported to persist much more robustly than influenza A under a range of environmental conditions [@doi:10.2807/1560-7917.es2013.18.38.20590; @doi:10.1186/s12985-015-0439-5].
While droplet-based and contact transmission were initially considered to be the primary modes by which SARS-CoV-2 spread [@doi:10.1038/s41368-020-0075-9], as additional information emerged, the possibility of aerosol transmission was also raised [@doi:10.1126/science.abc6197; @doi:10.1056/NEJMc2004973; @doi:10.1093/cid/ciaa939].
For example, the detection of SARS-CoV-2 viral particles in air samples taken from hospitals treating COVID-19 patients led to the concern that the virus could also spread via aerosols [@doi:10.1038/s41586-020-2271-3].
The stability of the virus both in aerosols and on a variety of surfaces appeared similar to that of SARS-CoV [@doi:10.1056/NEJMc2004973], and fomite transmission may play a role in transmission (e.g., [@doi:10.3201/eid2606.200412]).
However, while the possibility of aerosol transmission seems plausible, the evidence suggests that droplet and contact transmission are the dominant mechanisms driving the spread of the virus [@doi:10.1001/jama.2020.12458], and the risk of fomite transmission under real-world conditions is likely to be substantially lower than the conditions used for experimental analyses [@doi:10/gg6br7].
These type of transmission are likely to contribute to varying extents to different types of transmission events, such as transmission within households, nosocomial transmissions, and transmission in indoor versus outdoor spaces.

#### Symptoms and Viral Spread

Other aspects of pathogenesis are also important to understanding how the virus spreads, especially the relationship between symptoms, viral shedding, and contagiousness. 
Symptoms associated with reported cases of COVID-19 range from mild to severe [@url:https://www.cdc.gov/coronavirus/2019-ncov/symptoms-testing/symptoms.html], but some individuals who contract COVID-19 remain asymptomatic throughout the duration of the illness [@doi:10.1007/s11427-020-1661-4].
The incubation period, or time period between exposure and the onset of symptoms, has been estimated at five to eight days (4.91 and 7.54 in two different cities) [@doi:10.7554/eLife.57149; @doi:10.1164/rccm.202003-0524LE], and estimates suggest that viral shedding may begin long before the onset of symptoms (12.3 days with a 95% CI of 5.9 - 17.0) and peak around the onset of symptoms [@doi:10.1038/s41591-020-0869-5].
As these trends became apparent, concerns arose due to the potential for individuals who did not or did not yet show symptoms to transmit the virus [@url:https://www.cdc.gov/coronavirus/2019-ncov/prevent-getting-sick/how-covid-spreads.html].
Recovered individuals may also be able to transmit the virus after their symptoms cease.
Estimates of the communicable period based on twenty-four individuals who tested positive for SARS-CoV-2 prior to or without developing symptoms estimated that individuals may be contagious for one to twenty-one days, but they note that this estimate may be low [@doi:10.1007/s11427-020-1661-4].
Initially, viral nucleic acids were reported to remain at observable levels in the respiratory specimens of recovering hospitalized COVID-19 patients for a median of 20 days and with a maximum observed duration through 37 days, when data collection for the study ceased [@doi:10/ggnxb3].
As more estimates of the duration of viral shedding are released, they are beginning to converge around approximately three weeks from first positive PCR test and/or onset of symptoms (which, if present, are usually identified within three days of the initial PCR test).
For example, viral shedding was observed for up to 28 days following symptom onset in a second study [@doi:10.1038/s41586-020-2196-x] and for one to 24 days from first positive PCR test with a median of 12 days [@doi:10.1001/jama.2020.3204].
On the other hand, almost 70% of patients were reported to still have symptoms at the time that viral shedding ceased, although all symptoms reduced in prevalence between onset and cessation of viral shedding (CVS) [@10.1016/j.jcv.2020.104577].
They also reported that the median time that elapsed between the onset of symptoms and CVS was 23 days and between first positive PCR test and CVS was 17 days [@10.1016/j.jcv.2020.104577]. 
The fact that this study reported symptom onset to predate the first positive PCR test by an average of three days, however, suggests that there may be some methodological differences between it and related studies.
An analysis of residents of a nursing home with a known SARS-CoV-2 case measured similar viral load in residents who were asymptomatic regardless of whether they later developed symptoms, and the load in the asymptomatic residents was comparable to that of residents who displayed either typical of atypical symptoms [@doi:10.1056/NEJMoa2008457].
Taken together, these results indicate that the presence or absence of symptoms are not reliable predictors of viral shedding or of SARS-CoV-2 status (e.g, [@doi:10.1001/jama.2020.6887].
However, viral shedding is not necessarily indicative of contagiousness, and this study also concluded that the risk of spreading the infection was low after ten days from the onset of symptoms, as they determined the viral load in sputum was unlikely to pose a significant risk based on their efforts to culture samples _in vitro_ [@doi:10.1038/s41586-020-2196-x].

The extent to which asymptomatic (or presymptomatic) individuals are able to transmit SARS-CoV-2 has been a question of significant interest in the first six months of the pandemic.
Early reports (February and March 2020) emerged describing transmission from presymptomatic SARS-CoV-2-positive individuals to close family contacts [@doi:10.1001/jama.2020.2565; @doi:10.1016/j.ijid.2020.03.027].
One of these reports [@doi:10.1016/j.ijid.2020.03.027] also included a description of an individual who tested positive for SARS-CoV-2 but never developed symptoms.
Later analyses have sought to estimate the proportion of infections that could be traced back to a presymptomatic or asymptomatic individual.
Estimates of the proportion of individuals with asymptomatic infections have varied widely.
The proportion of asymptomatic individuals on board the Diamond Princess cruise ship, which was the site of an early COVID-19 outbreak, was estimated at 17.9% [@doi:10.2807/1560-7917], 
In contrast, a model using the prevalence of antibodies among residents of Wuhan, China estimated a much higher of asymptomatic cases, at approximately 7 in 8, or 87.5% [@doi:10.1093/pcmedi/pbaa032/5901532].
An estimate of the population of care homes in London found that, among the residents (median age 85), the rate of asymptomatic infection was 43.8%, and among the caretakers (median age 47), the rate was 49.1% [@doi:10.1016/j.eclinm.2020.100533]. 
The duration of viral shedding may also be longer in individuals with asymptomatic cases of COVID-19 compared to those who do show symptoms [@doi:10.1038/s41591-020-0965-6].
As a result, the potential for individuals who do not know they have COVID-19 to spread the virus is of significant concern.
In Singapore and Tianjin, two cities studied to estimate incubation period, an estimated 40-50% and 60-80% of cases respectively, were estimated to be caused by contact with an asymptomatic individuals [@doi:10.7554/eLife.57149].
An analysis of viral spread in the Italian town of Vo', which was the site of an early COVID-19 outbreak, revealed that 42.5% of cases were asymptomatic and that the rate was similar across age groups [@doi:10.1038/s41586-020-2488-1].
They argued that the town's lockdown was imperative for controlling the spread of COVID-19 because it isolated the asymptomatic individuals.
While more models are likely to emerge to better explore the effect of asymptomatic individuals on SARS-CoV-2 transmission, these results suggest that strategies for identifying and containing asymptomatic but contagious individuals are important for managing community spread. 

#### Estimating the Fatality Rate

Estimating the occurrence of asymptomatic and mild COVID-19 cases is important to identifying the mortality rate associated with COVID-19.
The mortality rate of greatest interest would be the total number of fatalities as a fraction of the total number of people infected.
One metric reported is often case fatality rate (CFR), which simply compares the number of COVID-19 related deaths to the number of confirmed or suspected cases.
However, in locations without universal testing protocols, it is impossible to identify all exposed or all infected individuals because so many asymptomatic or mild cases go undetected.
Therefore, a more informative metric is the infection fatality rate (IFR), which compares the known deaths to the estimated number of cases.
Meta-analyses have produced estimates of global IFR ranging from as low as 0.1% to as high as 1.04% [@doi:10.1101/2020.05.03.20089854; @url:https://www.cebm.net/covid-19/global-covid-19-case-fatality-rates/; @doi:10.1101/2020.05.11.20098780; @doi:10.1101/2020.05.03.20089854].
All of these estimates note that IFR varies widely around the world.
Estimates of infection rates are becoming more feasible as more data becomes available for modeling and will be bolstered as serological testing becomes more common and available.

### Dynamics of Transmission

Disease spread dynamics can be estimated using R~0~, the basic reproduction number, and R~t~, the effective reproduction number.
Thus, accurate estimates are crucial to understanding the dynamics of infection and to predict the effects of different interventions.
R~0~ and the timescale of infection (measured by the infectious period and the exposed period) govern population-level epidemic dynamics, with R~0~ being one of the most important epidemiological parameters [@isbn:9780691116174].
R~0~ is the average number of new (secondary) infections caused by one infected person, assuming a wholly susceptible population [@doi:10.1111/j.1467-9574.1996.tb01482.x].
A simple mechanistic model used to describe infectious disease dynamics is a susceptible-infected-recovered (SIR) compartmental model.
In this formulation individuals move through three states: susceptible, infected, and recovered; two parameters, $\gamma$ and $\beta$, specify the rate at which the infectious recover, and the infection transmission rate, respectively.
In this simple formulation, R~0~ is estimated as the ratio of $\beta$ and $\gamma$.[@doi:10.1126/science.abb5659; @isbn:9780691116174].
A pathogen can invade a susceptible population only if R~0~ > 1 [@isbn:9780691116174; @isbn:9780199209996].
The spread of an infectious disease at a particular time t can be quantified by R~t~, the effective reproduction number, which assumes that part of the population has already recovered (and thus gained immunity to reinfection) or that mitigating interventions were put into place.
For example, if only a fraction S~t~ of the population is still susceptible, R~t~ = S~t~ x R~0~.
When R~t~ is greater than 1, an epidemic grows (i.e., the proportion of the population that is infectious increases); when R~t~ is less than 1, the proportion of the population that is infectious decreases.
R~0~ and R~t~ can be estimated directly from epidemiological data or inferred using mathematical modeling.
Modeling approaches are typically based upon a classic epidemiological model structure: the SIR model and its extensions [@https://doi.org/10.1098/rspa.1927.0118; @doi:10.1038/280361a0].
In the context of SARS-CoV-2, more complex modified susceptible-exposed-infectious-recovered (SEIR) models are commonly used.[need citations]

Estimates of R~0~ for COVID-19 lie in the range R~0~=1.4-6.5 [@doi:10/ggjvr7; @doi:10.1093/jtm/taaa021; @doi:10.1126/science.abb3221].
Variation in R~0~ is expected between different populations and the estimated values of R~0~ discussed below are for specific populations in specific environments.
The different estimates of R~0~ should not necessarily be interpreted as a range of estimates of the same underlying parameter.

<!--I am not quite sure about this next paragraph. Most sophisticated models now incorporate "data" in their SEIR formulation, so I'm not sure it's accurate to draw the distinction.-->

In one study of international cases, the predicted value is R~0~=1.7 [@doi:10.1101/2020.03.21.20040329], in China (both Hubei province and nationwide), the value is predicted to lie in the range R~0~=2.0-3.6 [@doi:10.2139/ssrn.3524675; @doi:10.1101/2020.01.25.919787; @doi:10/ggjvr7], and on a cruise ship where an outbreak occurred, predicted R~0~=2.28 [@doi:10.1016/j.ijid.2020.02.033].
SEIR model-derived estimates of R~0~ range from 2.0 - 6.5 in China [@doi:10.3390/jcm9020462; @doi:10.1101/2020.01.27.20018952; @doi:10.1101/2020.01.23.916726; @doi:10.1101/2020.01.23.20018549] to R~0~=4.8 in France [@doi:10.1101/2020.03.22.20040915].
Using the same model as for the French population, a study estimated R~0~=2.6 in South Korea [@doi:10.1101/2020.03.22.20040915], which is consistent with other studies [@tag:Park2020_distancing].
From a meta-analysis of studies estimating R~0~, [@doi:10.1093/jtm/taaa021] predict the median as R~0~=2.79.

Inference of the effective reproduction number can provide insight into how populations respond to an infection, and the effectiveness of interventions.
In China, R~t~ was predicted to lie in the range 1.6-2.6 in Jan 2020, before travel restrictions [@doi:10/ggptcf].
R~t~ decreased from 2.35 one week before travel restrictions were imposed (Jan 23, 2020), to 1.05 one week after.
Using their model, the authors also estimate the probability of new outbreaks occurring: the probability of a single individual exporting virus causing a large outbreak is 17-25% assuming MERS-like or SARS-like transmission, and the probability of a large outbreak occurring after ≥4 infections exist at a new location is greater than 50%.
An independent study came to similar conclusions: in a two-week period before Jan 23 finding R~t~=2.38, and decreasing to R~t~ = 1.34 (using data from Jan 24 to Feb 3) or R~t~=0.98 (using data from Jan 24 to Feb 8) [@doi:10.1126/science.abb3221].
In South Korea, R~t~ was inferred for Feb-Mar 2020 in two cities: Daegu (the center of the outbreak), and Seoul [@tag:Park2020_distancing].
Metro data was also analyzed to estimate the effects of social distancing measures.
R~t~ decreased in Daegu from around 3 to <1 over the period that social distancing measures were introduced.
In Seoul, R~t~ decreased slightly, but remained close to 1 (and larger than R~t~ in Daegu).
This highlights that social distancing measures appeared to work to contain the infection in Daegu, but that in Seoul, R~t~ remains above 1, thus secondary outbreaks are possible.
It also shows the importance of region-specific analysis: the large decline in case load nationwide is mainly due to the Daegu region, and could hide persistence of the epidemic in other regions, such as Seoul and Gyeonggi-do.
In Iran, estimates of R~t~ declined from 4.86 in the first week to 2.1 by the fourth week after the first cases were reported [@doi:10.1101/2020.03.20.20038422].
In Europe, analysis of 11 countries inferred the dynamics of R~t~ over a time range from the beginning of the outbreak until March 28, 2020, by which point most countries had implemented major interventions (such as school closures, public gathering bans, and stay-at-home orders) [@doi:10.25561/77731].
Across all countries, the mean R~t~ before interventions began was estimated as 3.87; R~t~ varied considerably, from below 3 in Norway to above 4.5 in Spain.
After interventions, R~t~ decreased by an average of 64% across all countries, with mean R~t~=1.43.
The lowest predicted value was 0.97 for Norway and the highest was 2.64 for Sweden, which may be in part because Sweden did not implement social distancing measures on the same scale as other countries.
The study concludes that while large changes in R~t~ are observed, it is too early to tell whether the interventions put into place are sufficient to decrease R~t~ below 1.

More generally, population-level epidemic dynamics can be both observed and modelled [@doi:10.1126/science.abb5659].
Data and empirically determined biological mechanisms inform models, while models can be used to try to understand data and systems of interest or to make predictions about possible future dynamics, such as the estimation of capacity needs [@doi:10.1073/pnas.2004064117] or the comparison of predicted outcomes among prevention and control strategies [@doi:10/ggp3xq; @doi:10.1073/pnas.2004978117].
Many current efforts to model R~t~ have led to tools that assist the visualization of estimates in real time or over recent intervals [@url:https://epiforecasts.io/covid; @url:https://rt.live].
While these may be valuable resources, it is important to note that the estimates arise from models containing many assumptions and are dependent on the quality of the data they use, which varies widely by region.

### Molecular Signatures and Transmission

Genetic variation in SARS-CoV-2 has been used to elucidate patterns over time and space.
Numerous mutations [@doi:10.1186/s12967-020-02344-6; @doi:10.1093/nsr/nwaa036; @doi:10.1016/j.meegid.2020.104351] observed in individual SARS-CoV-2 genome sequences can be used to trace transmission patterns and have provided insight during outbreak investigations. <!--move citations to later in sentence?-->
Similar mutations observed in several patients may indicate that the patients belong to the same transmission group.
The tracking of SARS-CoV-2 mutations is recognized as an essential tool for controlling outbreaks that may provide valuable insights into the paths of SARS-CoV-2's spread [@doi:10/d5mg].
Several studies used phylogenetic analysis to determine the source of local COVID-19 outbreaks in Connecticut (USA), [@doi:10.1016/j.cell.2020.04.021], the New York City area (USA) [@doi:10.1126/science.abc1917], and Iceland [@doi:10.1056/NEJMoa2006100].
There is an ongoing effort to collect SARS-CoV-2 genomes throughout the COVID-19 outbreak, and as of August 7, 2020, more than 78,000 genome sequences have been collected from patients. <!-- TODO: consider auto-updating this -->
The sequencing data can be found at GISAID [@https://www.gisaid.org/], NCBI [@https://www.ncbi.nlm.nih.gov/sars-cov-2/], and COVID-19 data portal [@https://www.covid19dataportal.org/].

### Factors Influencing Susceptibility

<!-- Merge with PR #638, which covers sex, age, comorbidities-->

#### Ancestry

A number of studies have suggested associations between individuals' racial and ethnic backgrounds and their COVID-19 risk.
In particular, Black Americans are consistently identified as carrying a higher burden of COVID-19 than white Americans [@doi:10.1377/hlthaff.2020.00598; @doi:10.15585/mmwr.mm6925e1], with differences in the rates of kidney complications from COVID-19 particularly pronounced [@doi:10.1016/j.kint.2020.05.006].
Statistics from a number of cities indicate significant discrepancies between the proportion of COVID-19 cases and especially deaths in Black Americans relative to their representation in the general population [@doi:10/ggv494].
In addition to Black Americans, disproportionate harm and mortality from COVID-19 has also been noted in Latino/Hispanic Americans and in Native American and Alaskan Native communities, including the Navajo nation [@doi:10.1001/jama.2020.8598; @doi:10.1136/bmj.m1483; @doi:10.1111/jrh.12451; https://www.nytimes.com/2020/04/09/us/coronavirus-navajo-nation.html?searchResultPosition=10; @url:https://www.nytimes.com/interactive/2020/07/05/us/coronavirus-latinos-african-americans-cdc-data.html; @doi:10.1093/tbm/ibaa055].
In Brazil indigenous communities likewise carry an increased burden of COVID-19 [@doi:10.1126/science.abc0073].
In the United Kingdom, nonwhite ethnicity (principally Black or South Asian) was one of several factors found to be associated with a higher risk of death from COVID-19 [@doi:10.1038/s41586-020-2521-4].

From a genetic standpoint, it is highly unlikely that ancestry itself predisposes individuals to contracting COVID-19 or to experiencing severe COVID-19 outcomes.
Examining human genetic diversity indicates variation over a geographic continuum, and that most human genetic variation is associated with the African continent [@doi:10.1038/ng1438].
African-Americans are also a more genetically divere group relative to European-Americans, with a large number of rare alleles and a much smaller fraction of common alleles identified in African-Americans [@doi:10.1146/annurev.genom.9.081307.164258].
Therefore, the idea that African ancestry (at the continent level) might convey some sort of genetic risk for severe COVID-19 contrasts with what is known about worldwide human genetic diversity [@doi:10.1126/science.abd4842].
The possibility for genetic variants that confer some risk or some protection remains possible, but has not been widely explored, especially at a global level.
Research in Beijing of a small number (n=80) hospitalized COVID-19 patients revealed an association between severe COVID-19 outcomes and homozygosity for an allele in the interferon-induced transmembrane protein 3 (IFITM3) gene, which was selected as a candidate because it was previously found to be associated with influenza outcomes in Chinese patients [@doi:10.1093/infdis/jiaa224].
However, genetic variants associated with outcomes within ancestral groups are far less surprising than genetic variants explaining outcomes between groups.
Alleles in _ACE2_ and _TMPRSS2_ have been identified that vary in frequency among ancestral groups [@doi:10.1186/s12916-020-01673-z], but whether these variants are associated with COVID-19 susceptibility has not been explored.

Instead, examining patterns of COVID-19 susceptibility on a global scale that suggest that social factors are of primary importance in predicting mortality.
Reports from several sub-Saharan African countries have indicated that the effects of the COVID-19 pandemic have been less severe than expected based on the outbreaks in China and Italy.
In Kenya, for example, estimates of national prevalence based on testing blood donors for SARS-CoV-2 antibodies were consistent with 5% of Kenyan adults having recovered from COVID-19 [@doi:10.1101/2020.07.27.20162693].
This high seroprevalence of antibodies lies in sharp contrast to the low number of COVID-19 fatalities in Kenya, which at the time was 71 out of 2093 known cases [@doi:10.1101/2020.07.27.20162693].
Likewise, a serosurvey of health care workers in Blantyre City, Malawi reported an adjusted antibody prevalence of 12.3%, suggesting that the virus had been circulating more widely than thought and that the death rate was up eight times lower than models had predicted [@doi:10.1101/2020.07.30.20164970].
While several possible hypotheses for the apparent reduced impact of COVID-19 on the African continent are being explored, such as young demographics in many places [@doi:10.1126/science.369.6505.756], these reports present a stark contrast to the severity of COVID-19 in Americans and Europeans of African descent.
Additionally, ethnic minorities in the United Kingdom also tend to be younger than white British living in the same areas, yet the burden of COVID-19 is still more serious for minorities, especially people of Black Carribean ancestry, both in absolute numbers and when controlling for age and location [@url:https://www.ifs.org.uk/inequality/chapter/are-some-ethnic-groups-more-vulnerable-to-covid-19-than-others/].
Furthermore, the groups in the United States and United Kingdom that have been identified as carrying elevated COVID-19 burden, namely Black American, indigenous American, and Black and South Asian British, are quite distinct in their position on the human ancestral tree.
What is shared across these groups is instead a history of disenfranchisement under colonialism and ongoing systematic racism. 
A large analysis of over 11,000 COVID-19 patients hospitalized in 92 hospitals across U.S. states revealed that Black patients were younger, more often female, more likely to be on Medicaid, more likely to have comorbidities, and came from neighborhoods identified as more economically deprived than white patients [@doi:10.1001/jamanetworkopen.2020.18039].i
This study reported that when these factors were accounted for, the differences in mortality between Black and white patients were no longer significant.
Thus, the current evidence suggests that the apparent correlations between ancestry and health outcomes must be examined in the appropriate social context.

<!-- Systemic factors to follow -->
